<!DOCTYPE html>

<html lang="en">
  <head>
    <meta charset="utf-8">
    <title>BP205</title>
    <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no">
    <meta name="description" content="This is the official web page for the James Fraser Lab at UCSF.">
    <meta name="theme-color" content="#7fab91">
    <link href="/static/css/bootstrap.css" rel="stylesheet">
    <link href="/static/css/bootstrap-responsive.css" rel="stylesheet">
    <link href="/static/css/syntax.css" rel="stylesheet">
    <link href="/static/css/custom.css" rel="stylesheet">
    
    <script type="text/javascript" src="https://www.google.com/jsapi"></script>
    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.1/jquery.min.js"></script>
    <!-- HTML5 shim, for IE6-8 support of HTML5 elements -->
    <!--[if lt IE 9]>
      <script src="http://html5shim.googlecode.com/svn/trunk/html5.js"></script>
    <![endif]-->
  </head>

  <body>

    <!-- Static navbar -->
      <div class="navbar navbar-fixed-top navbar-default" role="navigation">
        <div class="container-fluid">
          <div class="navbar-header">
            <button type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-collapse">
              <span class="sr-only">Toggle navigation</span>
              <span class="icon-bar"></span>
              <span class="icon-bar"></span>
              <span class="icon-bar"></span>
            </button>
            <a class="navbar-brand" href="/">Fraser Lab @ UCSF</a>
          </div>
          <div class="navbar-collapse collapse">
            <ul class="nav navbar-nav navbar-right">
              
                
                  <li><a href="/" alt="home"> home</a></li>
                
              
                
                  <li><a href="/members" alt="people"> people</a></li>
                
              
                
                  <li><a href="/research" alt="projects"> projects</a></li>
                
              
                
                  <li><a href="/publications" alt="publications"> publications</a></li>
                
              
              <!-- <li><a href="https://sites.google.com/a/fraserlab.com/ucsf-private/" alt="Lab Private Area">private</a></li> -->
            </ul>
          </div><!--/.nav-collapse -->
        </div><!--/.container-fluid -->
      </div>

    <div class="container">


<h1 id="biophysics-205a-physical-underpinnings-of-biological-systems">Biophysics 205A: Physical Underpinnings of Biological Systems</h1>

<h2 id="fall-2015-syllabus">Fall 2015 Syllabus</h2>

<p><strong>Course Title:</strong> Physical Underpinnings of Biological Systems (PUBS)</p>

<p><strong>Course Credit:</strong> 4 units</p>

<p><strong>Course Format:</strong> 12 hours of lab per week</p>

<p><strong>Location:</strong> Genentech Hall Teaching Lab - Room 227</p>

<p><strong>Prerequisites:</strong> All incoming first year iPQB and CCB graduate students are required to enroll in this course.</p>

<p>Although TAs, instructors, and your fellow students will be happy to help out, it is important to be familiar with basic scripting and the principles of python PRIOR to starting the class.</p>

<p>This will be covered in bootcamp, but it is also a good idea to get a jump start during the summer (suggestions are listed below).</p>

<p><strong>Grading:</strong> Letter grade</p>

<p><strong>Textbook:</strong> None. Lab protocols and course materials will be available in class or online</p>

<p><strong>Course Days/Hours:</strong> Monday, Tuesday, Wednesday 1pm-5pm</p>

<p><strong>Instructors:</strong> James Fraser; <a href="mailto:jfraser+bp205a@fraserlab.com">jfraser+bp205a@fraserlab.com</a>; Martin Kampmann</p>

<p><strong>Course Coordinator:</strong> David Mavor; <a href="mailto:David.Mavor@ucsf.edu">David.Mavor@ucsf.edu</a></p>

<p><strong>Mass Spectrometry Coodinator:</strong> Danielle Swaney; <a href="mailto:Danielle.Swaney@ucsf.edu">Danielle.Swaney@ucsf.edu</a></p>

<p><strong>TAs:</strong></p>

<ul>
  <li><a href="mailto:Ina.Chen@ucsf.edu">Ina Chen</a></li>
  <li><a href="mailto:green.evan@gmail.com">Evan Green</a></li>
  <li><a href="mailto:lkenner@fraserlab.com">Lillian Kenner</a></li>
  <li><a href="mailto:brukmensa@gmail.com">Bruk Mensa</a></li>
  <li><a href="mailto:Leanna.Morinishi@ucsf.edu">Leanna Morinishi</a></li>
  <li><a href="mailto:Erin.Poss@ucsf.edu">Erin Poss</a></li>
</ul>

<p><strong>Lecturers/Facilitators:</strong></p>

<p>James Fraser, David Mavor, Danielle Swaney, Joe DeRisi, Martin Kampmann, Hiten Madhani, David Morgan, Jason Gestwicki, Eric Chow, Nadav Ahituv, Ryan Hernandez, Bo Huang, Justin Biel, Tanja Kortemme, Kyle Barlow</p>

<p><strong>Important Dates:</strong></p>

<ul>
  <li>Mass Spectrometry Presentation: Monday, October 12th</li>
  <li>NSF GRFP due Monday, October 26 (Life Sciences) and Friday, October 30 (Chemistry)</li>
  <li>Final presentations: Tuesday, November 24th</li>
</ul>

<p><strong>Background:</strong></p>

<p>“Precision Medicine” is an emerging theme in biomedical research and patient care, and refers to the use of genome-wide information, such as DNA sequence, expression profiling, metabolic labeling/imaging, and other technologies to better inform and ultimately customize therapy. For cancer medicine, discreet genomic changes can be tied directly to particular treatments, such as immunotherapies or small molecules directed against a mutated enzyme. However, the cancer genome is not necessarily a static entity, and may be subjected to intense selective pressures resulting in highly dynamic changes that manifest as relevant phenotypes, such as drug resistance or metastatic potential. Technological revolutions, such as DNA microarrays, followed by ultra-deep sequencing, have allowed high-resolution views of the genome and dynamical views of the expression programs they exhibit.</p>

<p>Despite the promise for personalized care, many challenges remain. The genome, its expression, and its translation into phenotype embody a highly complex and dynamic system, whether it is a cancer cell, a yeast cell, or even a virus. Mutations that drive a phenotype may not be necessarily distinguishable from those that are mere passengers, and the molecular determinants of large-scale alterations remain largely uncharacterized.</p>

<p>Ultimately, the goal is the synthesis of predictive models that can reveal fundamental regulatory principles, and in the case of patients, deliver actionable information for treatment, early detection, and prevention.</p>

<p><strong>Course Description:</strong></p>

<p>The course is a hands-on, project-based course that integrates proteomics, deep mutational profiling <a href="http://www.ncbi.nlm.nih.gov/pubmed/25075907">(Fowler and Fields, <em>Nature Methods</em>, 2014)</a>, and computational biology. The model organism, <em>Saccharomyces cerevisiae</em>, will be used as the organismal basis. Our goals are to experimentally determine the kinases responsible for phosphorylating ubiquitin and to measure the fitness of all possible individual point mutants of ubiquitin. Ubiquitin is an essential protein that is a key cellular integrator of stress, under a variety of experimental perturbations. The library of these point mutants was assembled by <a href="http://profiles.umassmed.edu/profiles/display/133553">Dan Bolon</a> and verified during a summer visit to the Fraser lab. The course is organized around modules, described below. Each hands-on module will be accompanied by lectures (either “chalk talk” or with slides). Students will present short talks to the class covering the assigned protocols or summarizing literature related to the class.   In addition students are expected to conduct their own literature reviews during the course of the project. Students will work in small teams, and each team will be assigned a different set of kinases for initial analysis. Based on the results, the teams will design a perturbation experiment for deep sequencing.  Students are expected to remain in their teams for the duration of the course, although team-team collaboration is highly encouraged. All team members are expected to participate in each activity.</p>

<p>At the conclusion of the mass spec module and at the end of the class, each team will orally present their findings to the class and faculty, limited to 15 minutes and 15 slides maximum, with 10 minutes for discussion and questions. All members of the team are expected to speak and describe their contributions. These presentations are currently scheduled for Oct 12th and Nov 24th, the final day of class.</p>

<p>Activities and speakers for each week will be announced at the beginning of each module.</p>

<p><strong>Course Goals:</strong></p>

<p>The goal of the course is to provide an immersive, hands-on experience in the context of genuine research questions. As articulated by <a href="http://www.sciencemag.org/content/338/6114/1542.long">Vale and colleagues</a>, there are tremendous advantages when graduate students work “pursuing a research question with unknown answers and uncertain outcomes, students and faculty combine their wits and skills to design experiments, evaluate progress, and troubleshoot along the way”. These advantages are likely to be common accross <a href="http://blogs.kqed.org/mindshift/2014/09/can-project-based-learning-close-gaps-in-science-education/">all learning levels</a>. In our course, teams may use whatever literature, software, and resources that are available publicly, and are encouraged to write their own scripts and software where necessary.</p>

<p>The “official” language of the class is <a href="https://www.python.org">python</a> - beginners should try <a href="http://learnpythonthehardway.org/book/">Learn Python The Hard Way</a>, people with a background in other languages should try <a href="https://developers.google.com/edu/python/">Google’s python course</a>. The QB3 Berkeley <a href="http://intro-prog-bioinfo-2014.wikispaces.com/">intensive python course</a> provides many biological examples.</p>

<p><a href="https://docs.google.com/spreadsheets/d/1BjKsN0B1hqd4dJW5slZ5KPuToCjSMRyA7Bl8MwWrbS4/edit#gid=0">Spreadsheet</a> with a listing of multiple Python resources</p>

<p>Students should be comfortable with basic syntax and scripting prior to the start of instruction.</p>

<p><strong>Although TAs, instructors, and your fellow students will be happy to help out, it is important to be familiar with basic scripting and the principles of python PRIOR to starting the class.</strong></p>

<p>In <strong>module 1</strong>, teams will be given a yeast strain with a single protein kinase gene deletion. Mass spectrometry will be used to identify changes in the phosphoproteome and, in particular, to evaluate any changes in phosphorylation of the protein Ubiquitin.</p>

<p>In <strong>module 2</strong>, each team will select a chemical perturbation. Using deep mutational profiling, each team will measure the fitness of all possible individual point mutants of Ubiquitin, an essential protein that is a key cellular integrator of stress. Upon processing the sequencing data, each team will perform comparisons against a reference dataset.</p>

<p>In <strong>module 3</strong>, the teams will leverage interactions with Tanja Kortemme and members of her lab to perform protein design protocols to predict sequences optimized for multiple criteria. They will test how well their deep sequencing data matches with protein design profiles generated under different constraints such as: protein stability or maintaining interactions with specific mediators of stress responses.</p>

<p><strong>Student Learning Objectives</strong></p>

<ul>
  <li>Laboratory safety</li>
  <li>Appropriate methods for documenting laboratory procedures</li>
  <li>Library prep and Ultra-Deep Sequencing</li>
  <li>Bioinformatics and algorithms</li>
  <li>Python scripting</li>
  <li>Experimental design and planning</li>
  <li>Yeast molecular biology</li>
  <li>Introduction to mass spectrometry</li>
</ul>

<p><strong>Class Policies</strong></p>

<p><em>Absences</em>: The instructor must be notified by the second week of classes for any planned absences, or in advance of class due to illness. Active participation in the laboratory is essential and students are required to attend normal class hours. Attendance during all of the three required presentations is absolutely mandatory, except in cases of doctor-excused medical illness. Any class material or lecture that is missed will be the responsibility of the student. Written evaluations of each team and its members will be provided to the Graduate Tracking System for inclusion into the graduate record, and provided to oral committee members and thesis committee members.</p>

<p><em>Accommodations for students with disabilities:</em> The Graduate Division embraces all students, including students with documented disabilities. UCSF is committed to providing all students equal access to all of its programs, services, and activities. Student Disability Services (SDS) is the campus office that works with students who have disabilities to determine and coordinate reasonable accommodations. Students who have, or think they may have, a disability are invited to contact SDS (StudentDisability@ucsf.edu); or 415-476-6595) for a confidential discussion and to review the process for requesting accommodations in classroom and clinical settings. More information is available online
at http://sds.ucsf.edu. Accommodations are never retroactive; therefore students are encouraged to register with Student Disability Services (http://sds.ucsf.edu/) as soon as they begin their programs. UCSF encourages students to engage in support seeking behavior via all of the resources available through Student Life, for consistent support and access to their programs.</p>

<h2 id="teams">Teams</h2>

<ul>
  <li>APUBSCRAWL: Sergei, Rachel, David (TA: Bruk, SWE1, K6, Cerulenin - 4.5 uM)</li>
  <li>ONION: Peter, Pooja, Nathan (TA: Ina, ATG1, K48, rapamycin - 200 nM)</li>
  <li>ET0H: Ruilin, Tamas, Fatima (TA: Lillian, KIN3, K63, menadione - 500 uM)</li>
  <li>PYND: Douglas, Yuliya, Paul, Nicholas (TA: Erin, ALK1, K29, 5-fluorocytosine 100 ug/mL)</li>
  <li>SHMOO: Nadja, Ryan, Daniel, Derek (TA: Leanna, CMK1, K11, CaCl2, 500 mM)</li>
  <li>WHANGEE: Cole, Alex, Emily, Charlotte (TA: Evan, TPK1, N-term, Tunicamycin 1ug/mL)</li>
</ul>

<h2 id="lab-work-individual-presentation-schedule-and-recommended-reading">Lab work, individual presentation schedule, and recommended reading</h2>

<p><strong>Week 1 – Theme: Ubiquitin and Programming</strong></p>

<p><em>Lab work: Pickle project and Yeast transformation</em></p>

<p>Lecturers: James Fraser (9/22), Joe DeRisi (9/22), Danielle Swaney (9/23)</p>

<p>Student Presentations: Yuliya (Transformation Lecture, 9/23)</p>

<p>Recommended reading:</p>

<ul>
  <li><a href="http://www.ncbi.nlm.nih.gov/pubmed/25075907">(Fowler and Fields, 2014)</a></li>
  <li><a href="http://www.ncbi.nlm.nih.gov/pubmed/23028185">(Finley et al., 2012)</a></li>
  <li><a href="http://www.ncbi.nlm.nih.gov/pubmed/26268526">(Herhaus and Dikic, 2015)</a></li>
</ul>

<p>Files for Computation:</p>

<ul>
  <li><a href="https://drive.google.com/file/d/0Bz5C8aG_xj4sc3g5V29LcXluSE0/edit?usp=sharing">Allele_dic.pkl</a></li>
  <li><a href="https://drive.google.com/file/d/0Bz5C8aG_xj4sRVZVMU9RdUJoRkk/edit?usp=sharing">translate.pkl</a></li>
  <li><a href="https://drive.google.com/file/d/0Bz5C8aG_xj4sa0dlbGczQnd5R3c/edit?usp=sharing">aminotonumber.pkl</a></li>
</ul>

<p>Other Class Material:</p>

<ul>
  <li><a href="/pubs/server/">Information about the server</a></li>
  <li><a href="https://drive.google.com/file/d/0Bx0d95RwVYufQkxnMmFfS3EzRDQ/view?usp=sharing">PDF of Lecture 1</a></li>
  <li><a href="https://drive.google.com/file/d/0Bx0d95RwVYufaEpLOVZTSFpuQTQ/view?usp=sharing">PDF of Lecture 2</a></li>
  <li><a href="https://instruct.uwo.ca/biology/3596a/startedyeast.pdf">Fred Sherman’s “Getting started with Yeast”</a></li>
  <li><a href="https://docs.google.com/document/d/1-6-rbLosBYkAZI4EOcmwP8C2wDNiLE4OWvXjNS0Yfko/edit?usp=sharing">Transformation Protocol</a></li>
</ul>

<p>Chemical menu:</p>

<ul>
  <li><a href="http://www.sigmaaldrich.com/MSDS/MSDS/DisplayMSDSPage.do?country=US&amp;language=en&amp;productNumber=93755&amp;brand=FLUKA&amp;PageToGoToURL=http%3A%2F%2Fwww.sigmaaldrich.com%2Fcatalog%2Fproduct%2Ffluka%2F93755%3Flang%3Den">Tunicamycin</a></li>
  <li><a href="http://www.sigmaaldrich.com/MSDS/MSDS/PleaseWaitMSDSPage.do?language=&amp;country=US&amp;brand=SIGMA&amp;productNumber=S3256&amp;PageToGoToURL=http://www.sigmaaldrich.com/catalog/product/sigma/s3256?lang=en&amp;region=US">Spermine</a></li>
  <li><a href="http://www.sigmaaldrich.com/MSDS/MSDS/DisplayMSDSPage.do?country=US&amp;language=en&amp;productNumber=R0395&amp;brand=SIGMA&amp;PageToGoToURL=http%3A%2F%2Fwww.sigmaaldrich.com%2Fcatalog%2Fproduct%2Fsigma%2Fr0395%3Flang%3Den">rapamycin</a></li>
  <li><a href="http://www.sigmaaldrich.com/MSDS/MSDS/PleaseWaitMSDSPage.do?language=&amp;country=US&amp;brand=SIGMA&amp;productNumber=H3274&amp;PageToGoToURL=http://www.sigmaaldrich.com/catalog/product/sigma/h3274?lang=en&amp;region=US">hygromycin B</a></li>
  <li><a href="http://www.sigmaaldrich.com/MSDS/MSDS/PleaseWaitMSDSPage.do?language=&amp;country=US&amp;brand=ALDRICH&amp;productNumber=339350&amp;PageToGoToURL=http://www.sigmaaldrich.com/catalog/product/aldrich/339350?lang=en&amp;region=US">Nickle Chloride</a></li>
  <li><a href="http://www.sigmaaldrich.com/MSDS/MSDS/PleaseWaitMSDSPage.do?language=&amp;country=US&amp;brand=SIGMA&amp;productNumber=A8056&amp;PageToGoToURL=http://www.sigmaaldrich.com/catalog/product/sigma/a8056?lang=en&amp;region=US">3-Amino-1,2,4-triazole</a></li>
  <li><a href="http://www.sigmaaldrich.com/MSDS/MSDS/DisplayMSDSPage.do?country=US&amp;language=en&amp;productNumber=C4901&amp;brand=SIAL&amp;PageToGoToURL=http%3A%2F%2Fwww.sigmaaldrich.com%2Fcatalog%2Fproduct%2Fsial%2Fc4901%3Flang%3Den">Calcium dichloride</a></li>
  <li><a href="http://www.sigmaaldrich.com/MSDS/MSDS/PleaseWaitMSDSPage.do?language=&amp;country=US&amp;brand=SIGMA&amp;productNumber=C2389&amp;PageToGoToURL=http://www.sigmaaldrich.com/catalog/product/sigma/c2389?lang=en&amp;region=US">Cerulenin</a></li>
  <li><a href="http://www.sigmaaldrich.com/MSDS/MSDS/PleaseWaitMSDSPage.do?language=&amp;country=US&amp;brand=SIAL&amp;productNumber=403024&amp;PageToGoToURL=http://www.sigmaaldrich.com/catalog/product/sial/403024?lang=en&amp;region=US">Cobalt acetate</a></li>
  <li><a href="http://www.sigmaaldrich.com/MSDS/MSDS/PleaseWaitMSDSPage.do?language=&amp;country=US&amp;brand=SIGMA&amp;productNumber=M3512&amp;PageToGoToURL=http://www.sigmaaldrich.com/catalog/product/sigma/m3512?lang=en&amp;region=US">miconazole</a></li>
  <li><a href="http://www.sigmaaldrich.com/MSDS/MSDS/PleaseWaitMSDSPage.do?language=&amp;country=US&amp;brand=SIGMA&amp;productNumber=F5251&amp;PageToGoToURL=http://www.sigmaaldrich.com/catalog/product/sigma/f5251?lang=en&amp;region=US">p-Fluoro-DL-phenylalanine</a></li>
  <li><a href="http://www.sigmaaldrich.com/MSDS/MSDS/PleaseWaitMSDSPage.do?language=&amp;country=US&amp;brand=SIGMA&amp;productNumber=T5648&amp;PageToGoToURL=http://www.sigmaaldrich.com/catalog/product/sigma/t5648?lang=en&amp;region=US">tamoxifen</a></li>
  <li><a href="http://www.sigmaaldrich.com/MSDS/MSDS/PleaseWaitMSDSPage.do?language=&amp;country=US&amp;brand=SIGMA&amp;productNumber=UC280&amp;PageToGoToURL=http://www.sigmaaldrich.com/catalog/product/sigma/uc280?lang=en&amp;region=US">ketoconazole</a></li>
  <li><a href="http://www.sigmaaldrich.com/MSDS/MSDS/PleaseWaitMSDSPage.do?language=&amp;country=US&amp;brand=SIGMA&amp;productNumber=C6019&amp;PageToGoToURL=http://www.sigmaaldrich.com/catalog/product/sigma/c6019?lang=en&amp;region=US">clotrinazole</a></li>
  <li><a href="http://www.sigmaaldrich.com/MSDS/MSDS/PleaseWaitMSDSPage.do?language=&amp;country=US&amp;brand=ALDRICH&amp;productNumber=M57405&amp;PageToGoToURL=http://www.sigmaaldrich.com/catalog/product/aldrich/m57405?lang=en&amp;region=US">menadione</a></li>
  <li><a href="http://www.pro-lab.com/msds/stains_pl392_calcofluor_white_reagent%20_msds_eu.pdf">Calcofluor white</a></li>
  <li><a href="http://www.sigmaaldrich.com/MSDS/MSDS/PleaseWaitMSDSPage.do?language=&amp;country=US&amp;brand=ALDRICH&amp;productNumber=451665&amp;PageToGoToURL=http://www.sigmaaldrich.com/catalog/product/aldrich/451665?lang=en&amp;region=US">CuCl2</a></li>
  <li><a href="http://www.sigmaaldrich.com/MSDS/MSDS/PleaseWaitMSDSPage.do?language=&amp;country=US&amp;brand=SIGMA&amp;productNumber=F7129&amp;PageToGoToURL=http://www.sigmaaldrich.com/catalog/product/sigma/f7129?lang=en&amp;region=US">5-fluorocytosine</a></li>
  <li><a href="http://www.sigmaaldrich.com/catalog/product/sigma/sml0312?lang=en&amp;region=US">acivicin</a></li>
  <li><a href="http://www.sigmaaldrich.com/MSDS/MSDS/PleaseWaitMSDSPage.do?language=&amp;country=US&amp;brand=SIGMA&amp;productNumber=A9528&amp;PageToGoToURL=http://www.sigmaaldrich.com/catalog/product/sigma/a9528?lang=en&amp;region=US">amphotericin B</a></li>
</ul>

<p><strong>Week 2 – Theme: Mass Spectrometry</strong></p>

<p><em>Lab work: Biochemical enrichment of phosphopeptides</em></p>

<p>Lecturers: Hiten Madhani (9/29)</p>

<p>Student Presentations: Sergei (Lab work outline, 9/28), Ryan  (Journal Club on <a href="http://www.ncbi.nlm.nih.gov/pubmed/25527291">Wauer et al, 2015</a>, 9/28), Fatmia (Lab work outline, 9/29), Alex (Lab work outline, 9/30), Nathan (Journal Club on <a href="http://www.ncbi.nlm.nih.gov/pubmed/12872131">Peng et al, 2003</a>, 9/30)</p>

<p>Other Class Material:</p>

<ul>
  <li><a href="https://docs.google.com/document/d/1nmJMwx4d_977X4B_ne_sYiEVDjww12Xr73znOkMZv1Y/edit?usp=sharing">Lysis-Tryptic Digest protocol (9/28), Stop digest-lyophylize protocol (9/29), IMAC-lyophylize protocl (9/30)</a></li>
</ul>

<p><strong>Week 3 – Theme: Chemical Petrubations</strong></p>

<p><em>Lab work: Growth rate measurements</em></p>

<p>Lecturers:  David Mavor (10/5), David Morgan (10/6), Martin Kampmann (10/7)</p>

<p>Student Presentations: Emily (Drug Concentration protocol, 10/5), 1 representative per team: Rachel, Pooja, Tamas, Nick, Derek, Charlotte (Growth Rate analysis, 10/7), Daniel (Journal Club on <a href="http://www.ncbi.nlm.nih.gov/pubmed/18420932">Hillenmeyer et al, 2009</a>, 10/7)</p>

<p>Other Class Material:</p>

<ul>
  <li><a href="https://docs.google.com/document/d/12Az3DGJlL4jZ2Rx5Y-xOM5y-EFrUIxhF5lf0DuWwiFE/edit?usp=sharing">Drug Concentration Protocol</a></li>
  <li><a href="https://drive.google.com/file/d/0Bx0d95RwVYufcEkxSV9TcExPTmc/view?usp=sharing">David Morgan’s Lecture</a></li>
</ul>

<p>**Week 4 – Theme:  Mass Spec Bioinformatics **</p>

<p><em>Lab work: Analyze mass spec data</em></p>

<p>Recommended reading:</p>

<ul>
  <li><a href="http://www.ncbi.nlm.nih.gov/pubmed/19029910">MaxQuant software - Cox and Mann, 2008</a></li>
  <li><a href="http://www.ncbi.nlm.nih.gov/pubmed/22673903">Lundby et al, 2012</a></li>
</ul>

<p>Lecturers: Danielle Swaney (10/12), Jason Gestwicki (10/13), Eric Chow (10/14)</p>

<p>Student Presentations: Paul (Mass spec pipeline, 10/12 at end of class)</p>

<p>Other Class Material:</p>

<ul>
  <li><a href="https://drive.google.com/file/d/0Bx0d95RwVYufb2VNM2E0Vk1pbHM/view?usp=sharing">Sample Evidence File</a></li>
  <li><a href="https://drive.google.com/file/d/0Bx0d95RwVYufcjFZTnZTQTN1X0U/view?usp=sharing">Sample Table Guide</a></li>
  <li><a href="https://drive.google.com/file/d/0Bx0d95RwVYufbDhOM1RzSWkyTjQ/view?usp=sharing">Sample Protein Groups</a></li>
  <li><a href="https://drive.google.com/file/d/0Bx0d95RwVYufSjF4Z01uZnNIMWc/view?usp=sharing">Sample Phosphosites</a></li>
  <li><a href="https://drive.google.com/file/d/0B89bAF4_3DdILTR3WVNjZkd6NW8/view?usp=sharing">Danielle’s Lecture 10/12/15</a></li>
  <li><a href="https://drive.google.com/folderview?id=0B89bAF4_3DdIbzd1V1JxWkxKTGs&amp;usp=sharing">Jason’s Lecture plus notes (Thanks Sergei!)</a></li>
</ul>

<p>The real data:</p>

<ul>
  <li><a href="https://drive.google.com/file/d/0Bx0d95RwVYufVThRY1NfQzN5MVE/view?usp=sharing">PUBS 2015 MS files.zip</a></li>
</ul>

<p><strong>Week 5 – Theme: Growth Competitions</strong></p>

<p><em>Lab work: Harvest library of all Ubiquitin mutants</em></p>

<p>Lecturers:</p>

<p>Student Presentations: All students (Mass spec presentation, 10/19), Nadja (Library collection growth protocol, 10/19), Douglas (Journal Club on <a href="http://www.ncbi.nlm.nih.gov/pubmed/23376099">Roscoe et al., 2013</a>, 10/20), Cole (Journal Club on <a href="http://www.ncbi.nlm.nih.gov/pubmed/24862281">Roscoe et al, 2014</a>, 10/21)</p>

<p>Other Class Material:</p>

<ul>
  <li><a href="https://docs.google.com/document/d/1OIC1oyDUla72RJlt5a6kquhzMdSC1IuGUyPyztDxddc/edit?usp=sharing">Protocol for library growth</a></li>
  <li><a href="https://drive.google.com/file/d/0Bx0d95RwVYufNk5palhZYmg0WHM/view?usp=sharing">Illustrated Protocol</a></li>
</ul>

<p><strong>Week 6 – Theme: Preparation for sequencing</strong></p>

<h3 id="nsf-grfp-due-this-week">NSF GRFP due this week!</h3>

<p><em>Lab work: Molecular biology to isolate library</em></p>

<p>Lecturers: Nadav Ahituv (10/27)</p>

<p>Student Presentations: Rachel (Yeast Miniprep protocol, 10/26), Tamas (Barcode explanation and assigment to teams, 10/27), Pooja (PCR 1, gel extraction, and PCR 2, 10/27), David (PCR cleanup and QBIT, 10/28)</p>

<p>Other Class Material:</p>

<ul>
  <li><a href="https://docs.google.com/document/d/1zk_h_pS1Ikb1VfqSSx_ePHROVGu3dl55mJ_NSeYw3ss/edit?usp=sharing">Yeast Miniprep protocol</a></li>
  <li><a href="https://drive.google.com/file/d/0Bx0d95RwVYufLURGVWdpQnNYeUE/view?usp=sharing">Illustrated Yeast Miniprep protocol</a></li>
  <li><a href="https://docs.google.com/spreadsheets/d/1HyPya8RkLuFGjEq6HjMipmN_kXvoqX34Pae38CJtcLo/edit?usp=sharing">Team and Time Specific Barcodes</a></li>
  <li><a href="https://docs.google.com/document/d/1KUXhrxh-QxumFcgfjUVXTdYNZq_956ZQ11zpUSxtGR0/edit?usp=sharing">Library PCR protocol</a></li>
  <li><a href="https://drive.google.com/file/d/0Bx0d95RwVYufb2MtUml0SU5OWms/view?usp=sharing">Gel extraction protocol</a></li>
  <li><a href="https://docs.google.com/document/d/1US4qYUKqMY-s9H_ifFyWCCDQJfykd8Xmz4RwA8EBO7w/edit?usp=sharing">Barcode Addition PCR protocol</a></li>
  <li><a href="http://omegabiotek.com/store/wp-content/uploads/2013/09/D6492_D6493-Cycle-Pure-Kit-Combo-Omega.pdf">PCR cleanup protocol</a></li>
  <li><a href="https://drive.google.com/file/d/0Bx0d95RwVYufM0xhZ01USllneHM/view?usp=sharing">QBit Protocol</a></li>
  <li>Final Library Hand-off Instructions</li>
</ul>

<p><strong>Week 7 – Theme: Sequencing Data</strong></p>

<p><em>Lab work: Begin sequencing pipeline</em></p>

<p>Lecturers: Eric Chow (11/2), Joe DeRisi (11/2), Ina Chen+TAs (11/2), Ryan Hernandez (11/3), Bo Huang (11/4)</p>

<p>Student Presentations: Charlotte (Journal Club on <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=23041932">McLaughlin et al, 2012</a>, 11/3)</p>

<p>Other Class Material:</p>

<ul>
  <li><a href="https://www.dropbox.com/s/2l6x7mliya56lv1/Hernandez_PUBS.pdf?dl=0">Ryan Hernandez’s Lecture</a></li>
  <li><a href="https://drive.google.com/file/d/0B89bAF4_3DdIUzlUMHhmU1pQRkk/view?usp=sharing">Ina’s Unix101</a></li>
  <li><a href="https://external-production.codecademy.com/en/courses/learn-the-command-line/lessons/navigation/exercises/your-first-command?action=resume">Command line Refresher</a></li>
  <li><a href="https://docs.google.com/presentation/d/1GMJg5oZIrMgenp6726jASmwLrbFJkYwlJRf7B8nM9g8/edit?usp=sharing">Computation Refresher</a></li>
  <li><a href="http://biologylabs.utah.edu/jorgensen/wayned/ape/">A Plasmid Editor (APE)</a></li>
  <li><a href="https://drive.google.com/file/d/0Bx0d95RwVYufNzJTaGpDU2lOR0E/view?usp=sharing">APE file for Plasmid</a></li>
  <li><a href="https://drive.google.com/file/d/0Bx0d95RwVYufMUhpcFdqRnN2aG8/view?usp=sharing">APE file for PCR product</a></li>
  <li><a href="https://docs.google.com/document/d/12CGyjBaVTB1ncw00-yRyQl6Acga-LS7bfywvrpxLbNg/edit?usp=sharing">PCR Product Annotation</a></li>
  <li><a href="http://support.illumina.com/help/SequencingAnalysisWorkflow/Content/Vault/Informatics/Sequencing_Analysis/CASAVA/swSEQ_mCA_FASTQFiles.htm">Information about the format of your fastq files</a></li>
  <li><a href="/pubs/matplotlib_server/">Using Matplotlib on the Server</a></li>
  <li><a href="https://drive.google.com/file/d/0B89bAF4_3DdIUE04UFhYdTR1MFE/view?usp=sharing">Allele Dictionary With WT</a></li>
</ul>

<p><strong>Week 8 – Theme: Team Shuffles</strong></p>

<p><em>Lab work: Unify sequencing pipeline</em></p>

<p>Lecturers: Kyle Barlow and Justin Biel (11/10)</p>

<p>Student Presentations: Team 1 minute update: What does the signal look like, what is failing, what do you think you guys have done that is different/innovative/working better than anything? (Sergei, Nat, Fatima, Yuliya, Ryan, Alex) (11/9), Derek (Journal Club on <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=22547823">Pollack et al, 2012</a>, moved from 11/4 to 11/9) Nick (Journal Club on <a href="http://www.ncbi.nlm.nih.gov/pubmed/18556554">Lange et al, 2008</a>, 11/10)</p>

<p>Sub-groups:</p>

<ul>
  <li>Biological inference (Bruk)</li>
  <li>Data squeezing and scoring (Leanna)</li>
  <li>Reproducibility and statistics (Ina)</li>
  <li>Data visualization (Lillian)</li>
  <li>Structure painting (Erin)</li>
  <li>Comparing data sets (Evan)</li>
</ul>

<p>Other Class Material:</p>

<ul>
  <li><a href="http://en.wikipedia.org/wiki/Hamming_distance">Hamming Distance</a></li>
  <li><a href="https://drive.google.com/file/d/0B89bAF4_3DdIX0FibDVhazREeDBJeHNYeHBHVDI1U3NDdVdJ/view?usp=sharing">Kyle’s Lecture (ddG)</a></li>
  <li>
    <p><a href="https://drive.google.com/folderview?id=0B89bAF4_3DdId0FjdG1yb1pRbWc&amp;usp=sharing">Justin’s Pymol Demo</a></p>
  </li>
  <li>Reminder: no class on Nov 11</li>
</ul>

<p><strong>Week 9 – Theme: Biophysical Calcuations</strong></p>

<p><em>Lab work: Rosetta predictions of protein stability</em></p>

<p>Lecturers: Tanja Kortemme (11/16)</p>

<p>Student Presentations: Daniel, Emily, Paul and other representatives from Sub-groups (10/16), Ruilin (Journal Club on <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=23201676">Ye et al, 2012</a>, 10/17), Peter (Journal Club on <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=17722975">Humphris et al, 2007</a>, 10/18)</p>

<p>Other Class Material:</p>

<ul>
  <li>APUBSCRAWL: K6 linked poly-Ub - 2XK5</li>
  <li>ONION: K48 linked poly-Ub - 1AAR</li>
  <li>ET0H: K63 linked poly-Ub - 3H7P</li>
  <li>PYND: K29 linked poly-Ub - 4S22</li>
  <li>SHMOO: K11 linked poly-Ub - 3NOB</li>
  <li>WHANGEE: N-term linked poly-Ub - 2W9N</li>
  <li><a href="https://guybrush.ucsf.edu/local/DDG/ubiquitin">ddG Data Website</a></li>
</ul>

<p>#Tuesday November 24 - FINAL TEAM PRESENTATIONS</p>

<p>#15 slide maximum, 15 minutes per team (plus questions)</p>

<p>Schedule:</p>

<ul>
  <li>1:10-1:30 - SHMOO: Nadja, Ryan, Daniel, Derek (TA: Leanna)</li>
  <li>1:30-1:50 - WHANGEE: Cole, Alex, Emily, Charlotte (TA: Evan)</li>
  <li>1:50-2:10 - APUBSCRAWL: Sergei, Rachel, David (TA: Bruk)</li>
  <li>2:10-2:30 - ONION: Peter, Pooja, Nathan (TA: Ina)</li>
  <li>2:30-2:50 - PYND: Douglas, Yuliya, Paul, Nicholas (TA: Erin)</li>
  <li>2:50-3:10 - ET0H: Ruilin, Tamas, Fatima (TA: Lillian)</li>
  <li>3:10-… PUBS party</li>
</ul>


    </div>
    <hr>
    <div id="footer footer-fixed-bottom">
      <div class="container">
        <p class="text-muted credit"> This website is made using the template of <a href="https://github.com/fraser-lab/fraser-lab.github.io">the Fraser Lab</a>.</p>
      </div>
    </div>

     <!-- Bootstrap core JavaScript
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    
    <script src="/static/js/bootstrap.min.js"></script>
    
    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
    
      ga('create', 'UA-52773332-1', 'auto');
      ga('send', 'pageview');
    
    </script>
  </body>
</html>
