<!DOCTYPE html>

<html lang="en">
  <head>
    <meta charset="utf-8">
    <title>News from the Fraser Lab</title>
    <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no">
    <meta name="description" content="This is the official web page for the James Fraser Lab at UCSF.">
    <meta name="theme-color" content="#7fab91">
    <link href="/static/css/bootstrap.css" rel="stylesheet">
    <link href="/static/css/bootstrap-responsive.css" rel="stylesheet">
    <link href="/static/css/syntax.css" rel="stylesheet">
    <link href="/static/css/custom.css" rel="stylesheet">
    
    <link rel="alternate" type="application/atom+xml" title="Fraser Lab News Feed" href="/news.xml">
    
    <script type="text/javascript" src="https://www.google.com/jsapi"></script>
    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.1/jquery.min.js"></script>
    <!-- HTML5 shim, for IE6-8 support of HTML5 elements -->
    <!--[if lt IE 9]>
      <script src="http://html5shim.googlecode.com/svn/trunk/html5.js"></script>
    <![endif]-->
  </head>

  <body>

    <!-- Static navbar -->
      <div class="navbar navbar-fixed-top navbar-default" role="navigation">
        <div class="container-fluid">
          <div class="navbar-header">
            <button type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-collapse">
              <span class="sr-only">Toggle navigation</span>
              <span class="icon-bar"></span>
              <span class="icon-bar"></span>
              <span class="icon-bar"></span>
            </button>
            <a class="navbar-brand" href="/">Fraser Lab @ UCSF</a>
          </div>
          <div class="navbar-collapse collapse">
            <ul class="nav navbar-nav navbar-right">
              
                
                  <li><a href="/" alt="home"> home</a></li>
                
              
                
                  <li><a href="/research" alt="research"> research</a></li>
                
              
                
                  <li><a href="/publications" alt="publications"> publications</a></li>
                
              
                
                  <li><a href="/members" alt="members"> members</a></li>
                
              
                
                  <li class = "active"><a href="/news" alt="news"> news</a></li>
                
              
                
                  <li><a href="/contact" alt="contact"> contact</a></li>
                
              
                
                  <li><a href="https://sites.google.com/a/fraserlab.com/ucsf-private/" alt="private"> private</a></li>
                
              
              <!-- <li><a href="https://sites.google.com/a/fraserlab.com/ucsf-private/" alt="Lab Private Area">private</a></li> -->
            </ul>
          </div><!--/.nav-collapse -->
        </div><!--/.container-fluid -->
      </div>

    <div class="container">


<div class="row-fluid">
    <div class="col-sm-7 col-md-8">
        <div class = "page header"><h1>News from the Fraser Lab</h1></div>
        <!-- This loops through the paginated posts -->

  <hr>
  <h2><a href="/2015/04/22/Spring-quarter-has-sprung/">Spring quarter has sprung</a></h2>
  <p class="author">
    <span class="author"><a href="/members#James Fraser">James Fraser</a></span><br>
    <span class="date"><em>22 April 2015</em></span>
  </p>
  <div class="content">
    <p>A belated welcome to our new rotation student Eugene!</p>

<p><img src="/static/img/news/eugene.jpg" alt="Gene-Cakes" class="img-responsive" /></p>

<p>And farewell to Erin and Evan who have moved on to their final rotations in the Gestwicki and Frost labs, respectively…</p>

<p>We are apparently on a roll of rotation students with first names that start with the letter “E”…</p>

  </div>

  <hr>
  <h2><a href="/2015/02/18/EMringer/">EMRinger: a side-chain-directed approach to study model-to-map agreement in cryo-EM</a></h2>
  <p class="author">
    <span class="author"><a href="/members#Ben Barad">Ben Barad</a></span><br>
    <span class="date"><em>18 February 2015</em></span>
  </p>
  <div class="content">
    <p>For the past year, I have been working in the <a href="http://fraserlab.com">Fraser Lab</a> on developing an analysis framework, which we call <strong>EMRinger</strong>, for doing model-to-map validation in the burgeoning field of near-atomic-resolution single particle electron cryomicroscopy (cryo-EM). We recently submitted our paper for review; at the same time, we preprinted it in <a href="http://dx.doi.org/10.1101/014738">bioRxiv</a> and <a href="https://github.com/fraser-lab/EMRinger">open sourced the code</a>. We hope that even as the article undergoes the peer review process, the tool will be available for scientists hoping to get an independent metric for progress in their refinement. We also hope people will be able to start including it as a “Table 1” metric now that the code and the manuscript are available. We have already been using EMRinger with our collaborators for a few months, and we are excited to see how it gets used now that it is out in the wild!</p>

<p>The manuscript is the best place to get scientific details about the work, but I am writing this post to talk informally about the method, as well as the process of developing it and the preprinting experience.</p>

<!--break-->

<h2 id="the-background-ab-initio-modeling-in-cryo-em">The background: <em>ab initio</em> modeling in cryo-EM</h2>
<p>In the past two years, electron cryomicroscopy has been able to achieve resolutions at which atomic models can begin to be inferred without any sort of crystallographic reference structure, as a result of major developments both in camera technology and computational analysis. Electron microscopy holds tremendous promise for structural biology because of its ability to solve structures with minimal amounts of protein and without needing to find conditions for crystallization. Already, it has become one of the preferred methods for solving structures of large membrane proteins and biological complexes. However, the tools for validation of structures from cryo-EM are in their infancy relative to diffraction methods, and there are different requirements for validation compared to diffraction. Our tool takes advantage of the density around side chains, the highest resolution information available, to sensitively report on backbone positioning in the map.</p>

<!-- 
### Validation in cryo-EM
Because of the separate construction of the model and map in cryo-EM, three steps of validation are needed. First, there needs to be a check to ensure that the map is an accurate reconstruction of the images collected by the microscope. This has been a challenge for cryo-EM historically, but in recent years the gold standard FSC metric has become a common approach to solving this problem. In this method, the imaged particles are split in half and separately reconstructed, and then the two reconstructions are compared with fourier shell correlation, and the point at which they diverge significantly (FSC<0.143) is chosen as the maximum resolution. 

When atomic models are built *de novo* from high resolution maps, there must also be checks that the models are physically realistic. This is accomplished by checking the stereochemistry, ramachandran statistics, side chain rotamers, and steric clashes in three dimensions. A common tool used to accomplish this analysis is MolProbity. A model that is a perfect representation of a map but violates the physical expectations of proteins is likely to be overfitted to the microscopy data. An accurate model needs a balance between data from experiment and prior physical knowledge.

Because of the separation of model and map generation in cryo-EM, a third step of validation in required, in which the quality of the fit of the model in the map is quantified. This fit to density has been quantified historically in a few ways, but in general the tools are effective for reporting on the global fit of secondary structure in the map, but are unsuitable for quantifying the precise placement of atoms within those secondary structures. We sought to develop a tool that could much more sensitively report on atomic positioning in two ways. First, we used side chain information to report only on the highest sensitivity part of the model-to-map fit, and second, we leveraged our prior statistical knowledge about side chain geometry to gain additional power to report on fit. -->

<h2 id="emringer---using-side-chain-information-to-sensitively-report-on-backbone-position">EMRinger - using side chain information to sensitively report on backbone position.</h2>
<p><img src="/static/img/news/emringer.png" class="img-responsive center-block" alt="EMRinger Process" width="640" />
EMRinger is named after <a href="http://bl831.als.lbl.gov/ringer/">its progenitor, Ringer</a>, from the Alber lab at Berkeley. Like Ringer, it rotates the Cγ atom around the χ1 angle of a side chain, interpolating the density value in the map as it rotates. The angle of peak density is interpreted as the correct position of the side chain in dihedral space. Based on knowledge from performing Ringer on structures in the PDB, as well as from basic steric considerations, we know that properly modeled side chains should have χ1 density peaks at or near rotameric positions (60º, 180º, and 300º). There are two major reasons that the peak may not be rotameric: first, if there is no signal above noise from the side chain, the peak will be chosen randomly from the noise and may be rotameric, and second, if the backbone and Cβ positions are incorrect, the measured angle relative to that backbone may be reported as nonrotameric when the Cγ samples its true position. We take advantage of this second expected behavior to quantify the quality of model-to-map fit of models from cryo-EM.</p>

<h3 id="generating-the-emringer-score">Generating the EMRinger score</h3>
<p>In order to separate the effects of noise from the effects of mispositioned backbone atoms, we implement a signal cutoff threshold for peaks, with the assumption that any side chain with a peak map value below that threshold may be sampling noise and should not be used for analysis. For the remaining side chains, we use the fraction of side chains with rotameric peaks from EMRinger as a metric for the quality of model-to-map fit.</p>

<p>Choosing the right threshold is a challenging question; at too low of a cutoff, peaks are coming from noise as often as from real signal, and at too high of a cutoff, very few side chains are sampled and the results are not very representative of the model as a whole. In order to balance these two considerations, we use a Zscore metric based on the null hypothesis of a random binomial distribution with 39 out of 72 bins being called rotameric.</p>

<!-- <a href="http://www.codecogs.com/eqnedit.php?latex=Zscore_{threshold}=\frac{Number\&space;Rotameric_{threshold}&space;-&space;\frac{39}{72}\cdot&space;Sample\&space;Size_{threshold}}{\sqrt{\frac{33}{72}\cdot\frac{39}{72}\cdot&space;Sample\&space;Size_{threshold}}}" target="_blank"> -->
<!-- <img src="http://latex.codecogs.com/gif.latex?Zscore_{threshold}=\frac{Number\&space;Rotameric_{threshold}&space;-&space;\frac{39}{72}\cdot&space;Sample\&space;Size_{threshold}}{\sqrt{\frac{33}{72}\cdot\frac{39}{72}\cdot&space;Sample\&space;Size_{threshold}}}" title="Zscore_{threshold}=\frac{Number\ Rotameric_{threshold} - \frac{39}{72}\cdot Sample\ Size_{threshold}}{\sqrt{\frac{33}{72}\cdot\frac{39}{72}\cdot Sample\ Size_{threshold}}}" class="img-responsive center-block" alt="Zscore Equation"/>
 -->

<div class="row-fluid">

  <div class="col-xs-12 col-lg-10 col-lg-offset-1">

    <p><img src="/static/img/news/emringer_eq_1.svg" class="img-responsive center-block" alt="Zscore Equation" width="100%" /></p>

  </div>

  <div class="clearfix hidden-xs"></div>

</div>

<p>The Z-score is a good metric that is (in our experience) always able to find an ideal threshold to maximize the statistical significance of the result. However, it is biased towards models which are larger to begin with; a 60-mer would have a score more than 7 times larger than the equivalent monomer. To account for this, we rescale the Z-score to the <strong>EMRinger Score</strong>, which is comparable between sample sizes.</p>

<!-- <a href="http://www.codecogs.com/eqnedit.php?latex=EMRinger\&space;Score_{threshold}=\frac{10\cdot&space;Zscore_{threshold}}{\sqrt{Model\&space;Length}}" target="_blank"> -->
<!-- <img src="http://latex.codecogs.com/gif.latex?EMRinger\&space;Score_{threshold}=\frac{10\cdot&space;Zscore_{threshold}}{\sqrt{Model\&space;Length}}" title="EMRinger\ Score_{threshold}=\frac{10\cdot Zscore_{threshold}}{\sqrt{Model\ Length}}" class="img-responsive center-block" alt="EMRinger Score Equation"/>
 -->

<div class="row-fluid">

  <div class="col-xs-12 col-lg-10 col-lg-offset-1">

    <p><img src="/static/img/news/emringer_eq_2.svg" class="img-responsive center-block" alt="EMRinger Score Equation" width="100%" /></p>

  </div>

  <div class="clearfix hidden-xs"></div>

</div>

<p>The maximum EMRinger score obtained across threshold is the one we use as our final metric, and our evidence (as we discuss in the manuscript) is that it is a robust but sensitive metric for model-to-map fit in very high resolution maps, above around 4 Å. The EMRinger scores of model-map pairs from the EMDB sets up a reasonable expectation for what scores should look like for well-refined structures at high resolution:</p>

<script type="text/javascript">

// Load the Visualization API and the piechart package.
google.load('visualization', '1.0', {'packages':['corechart']});

// Set a callback to run when the Google Visualization API is loaded.
google.setOnLoadCallback(drawChart);

// Callback that creates and populates a data table,
// instantiates the pie chart, passes in the data and
// draws it.
function drawChart() {

// Create the data table.
var data = new google.visualization.DataTable();
data.addColumn('number', 'Resolution');
data.addColumn('number', 'EMRinger Score');
data.addColumn({type:'number',role:'annotation'})
data.addColumn({type:'string',role:'annotationText'})
data.addRows([
[3.1,	1.54,	5256, 'EMDB 5256 scores 1.54'],
[3.2,	2.04,	5995, 'EMDB 5995 scores 2.04'],
[3.2,	2.18,	5160, 'EMDB 5160 scores 2.18'],
[3.2,	3.05,	5623, 'EMDB 5623 scores 3.05'],
[3.27,	0.56,	5778, 'EMDB 5778 scores 0.56'],
[3.36,	1.29,	2513, 'EMDB 2513 scores 1.29'],
[3.4,	1.85,	2787, 'EMDB 2787 scores 1.85'],
[3.4,	2.09,	2762, 'EMDB 2762 scores 2.09'],
[3.5,	0.96,	6035, 'EMDB 6035 scores 0.96'],
[3.5,	1.95,	5764, 'EMDB 5764 scores 1.95'],
[3.5,	3.26,	2278, 'EMDB 2278 scores 3.26'],
[3.6,	1.23,	5925, 'EMDB 5925 scores 1.23'],
[3.75,	0.9	,	2764, 'EMDB 2764 scores 0.9'],
[3.8,	0.36,	2773, 'EMDB 2773 scores 0.36'],
[3.8,	1.05,	5830, 'EMDB 5830 scores 1.05'],
[3.8,	2.08,	6000, 'EMDB 6000 scores 2.08'],
[4,		0.54,	2763, 'EMDB 2763 scores 0.54'],
[4.1,	0.18,	5600, 'EMDB 5600 scores 0.18'],
[4.4,	-0.47,	2364, 'EMDB 2364 scores -0.47'],
[4.5,	-0.41,	2677, 'EMDB 2677 scores -0.41'],
[4.5,	0.13,	2273, 'EMDB 2273 scores 0.13'],
[4.5,	0.49,	5678, 'EMDB 5678 scores 0.49'],
[4.6,	-0.05,	5645, 'EMDB 5645 scores -0.0'],
[4.7,	0.09,	5895, 'EMDB 5895 scores 0.09'],
[4.7,	0.55,	5646, 'EMDB 5646 scores 0.55'],
[4.7,	1.27,	2788, 'EMDB 2788 scores 1.27'],
[4.9,	0.2	,	5391, 'EMDB 5391 scores 0.2'],
[5,		-0.71,	6187, 'EMDB 6187 scores -0.71'],
[5,		-0.16,	6188, 'EMDB 6188 scores -0.16'],
[5,		0.06,	5896, 'EMDB 5896 scores 0.06'],
[5,		0.8,	5886, 'EMDB 5886 scores 0.8']
]);

// Set chart options

var chartOptions = {
reverse: true,
displayAnnotations: true,
annotations: { 0: {
	highContrast: false,
	textStyle: {
		fontName: "Arial",
		fontSize: 5,
		opacity: 0.5
}}},
title: 'EMRinger Score is correlated to resolution',
titleTextStyle: {
	fontSize: 24
},
hAxis: {
title: 'Resolution (Å)',
direction: -1,
ticks: [3.0, 3.4, 3.8, 4.2, 4.6, 5.0]
},
vAxis: {
title: 'EMRinger Score',
viewWindow: {min: -1, max: 4},
ticks: [-1, 0, 1, 2, 3, 4]
},
trendlines: { 0: {
	type: 'linear',
	showR2: true,
	lineWidth: 2, 
	pointSize: 0,
	tooltip: false,
	visibleInLegend: true
}},
legend: {position: 'bottom', alignment: 'center'},
lineWidth: 0,
pointSize: 5
};



var chart = new google.visualization.LineChart(document.getElementById('chart_div'));
chart.draw(data, chartOptions);
}

$(window).resize(function(){
drawChart();
});

$(document).ready(function(){
drawChart();
});

</script>

<div id="chart_div" class="chart">

</div>

<p>There is a strong correlation between resolution and EMRinger score. This is to be expected, given that EMRinger score reports on side chain density, which is only resolvable above about 4.5 Å. In general, for maps above around 3.5 Å resolution, the minimum score that should be expected is around 1, with a benchmark for a very good score lying around 2. Most structures which have been carefully refined, either in real or reciprocal space, score above 1.5, with some structures getting scores above 3. Just like with tools like molprobity, a bad score doesn’t necessarily mean an unusable structure, but it does warrant further consideration and refinement.</p>

<p>At resolutions down near 4 Å and below, things get a little bit trickier, and it is hard to make careful predictions about what the target EMRinger score should be. Sampling of the backbone by EMRinger becomes a problem at these resolutions, which can be partially resolved through B-factor sharpening. We are considering ways to modify the EMRinger procedure to make it more appropriate for use in these intermediate resolutions in the future, and I will hopefully write more about that soon!</p>

<h2 id="making-emringer-scores-available">Making EMRinger Scores Available</h2>

<p>Since the score is straightforward to calculate and better scores are strong indicators of a better-fit model, we hope that this tool will be adopted as a standard “Table 1” metric for structures from cryo-EM, alongside gold-standard FSC resolution and Molprobity scores. To that end, we have made the tool available to colleagues as quickly as possible upon completing the manuscript: we have preprinted our work, and we have published and open sourced our code immediately.</p>

<h3 id="preprinting-with-biorxiv">Preprinting with bioRxiv</h3>

<p>The first and most important thing we did to increase the availability of EMRinger was to preprint the paper on <a href="http://biorxiv.org/content/early/2015/02/03/014738">bioRxiv</a> as soon as we submitted the manuscript for review. <a href="http://biorxiv.org/about-biorxiv">BioRxiv</a> is a service maintained by Cold Spring Harbor which allows the public deposition of a manuscript before, or even entirely separately from, the peer review process. This allows us to publically share the work we have been doing without waiting for the peer review process, which has been known to take over a year before publication. Additionally, people unrelated to the official peer review process, but who have suggestions for strengthening the method, can provide commentary on bioRxiv that we can then take into account when revising our paper. The <a href="http://fraserlab.com">Fraser Lab</a> has recently gotten excited about bioRxiv, and we plan on releasing many of our future papers this way. With many (most?) journals now allowing preprinting of manuscripts before peer review, there is little to lose in doing so. So far, we have primarily heard from colleagues directly about our manuscript, but we are excited to see what comments come out of the public forum available on bioRxiv.</p>

<p>We have been really happy with the reception of the paper on bioRxiv; we’ve had about 80 downloads of the paper in the two weeks since we submitted it. James and I have immediately become obsessed with the article metrics available on the site (particularly the altmetric). We are big <a href="http://twitter.com/fraser_lab">twitter</a> <a href="http://twitter.com/benjaminbarad">users</a> and we take every chance we get to share our progress on EMRinger now that it is available in a public form. More importantly, we’ve been able to share the method directly with our colleagues, and if they want to use the EMRinger Score in their publication they will be able to cite the manuscript on bioRxiv ahead of formal publication.</p>

<h3 id="publishing-code-on-github">Publishing code on Github</h3>

<p>Along with making the manuscript available, we have <a href="https://github.com/fraser-lab/EMRinger">released the code to run EMRinger on Github.</a> We have published the main utilities to calculate EMRinger score, perform rolling-window analysis, and confine EMRinger analysis to specific residue groups, which are the main ways we have used EMRinger and expect it to be used in the near future. I am also in the process of cleaning up some of my “jiffy” scripts to generate the figures we used in the manuscript, so that the connection between the output of the EMRinger code and the figures we present is as clear as possible. We hope this will also allow anyone interested in extending the usage of EMRinger into new domains to visualize their results quickly.</p>

<p>I wrote the EMRinger scripts in <a href="http://python.org">python</a> using the <a href="http://phenix-online.org">Phenix</a> software package, which provides a whole host of incredibly nice utilities for working with structural biology data. Phenix has so far been used mostly for data from diffraction experiments, but there are an increasing number of tools available for single particle experiments, and I am hopeful that Phenix will be able to bring together experimentalists from both fields. This has the potential to develop standardization in the tools for modeling and analysis, as well as in the data formats between the two fields.</p>

<p>I wrote a detailed usage guide for generating EMRinger Scores in the <a href="https://github.com/fraser-lab/EMRinger/tree/master/Phenix_Scripts"><code class="highlighter-rouge">Phenix_Scripts</code> folder of the EMRinger github page</a>, where the main utility scripts for EMRinger are housed. However, if you try it out, and encounter any problems that are not covered in the usage guide, or just want to discuss how best to interpret your results, I am happy to communicate <a href="mailto:ben.barad@ucsf.edu">by email</a>. Please submit bug reports or feature requests on the <a href="https://github.com/fraser-lab/EMRinger/issues">github</a>, and general comments on the biorxiv page!</p>

<h3 id="next-steps-phenix-gui-and-emringer-web-server">Next steps: Phenix GUI and EMRinger Web Server</h3>

<p>We are very happy with the Phenix python scripts on github, but in the long term we want to make our tool as universally accessible as possible, without any need to run scripts from the command line. We have been working hard on making EMRinger more accessible in two main ways. The first is putting the tool into the Phenix GUI: this effort is being driven primarily by our collaborator, Nathaniel Echols. Bringing EMRinger into the GUI will allow the rapid calculation of EMRinger scores, as well as making interactive visualization of the ringer scans possible. Since many other analysis tools, including molprobity, are accessible within Phenix, we hope that this inclusion will allow electron microscopists to do every level of model-based validation within a single software package, without needing to resort to individual scripts.</p>

<p>The second way we are trying to expand the availability of EMRinger is by making a webserver to calculate EMRinger scores on demand. I have been working on this as a side project for a few months, and actually got a proof of principle iteration done as part of a <a href="https://www.cgl.ucsf.edu/Outreach/bmi219/">minicourse at UCSF</a> with Kevin Hartman and Andrew Van Benschoten. Users will be able to upload their model and map and the webserver will return a series of visualizations as well as the final EMRinger score. I hope that this serves as the most accessible introduction possible to using EMRinger, as the only software needed will be a modern web browser. I’ll be open sourcing it as soon as I have a working prototype (soon, I hope!), and I’ll probably write another post then about the process of making a scientific webserver. There are unique scalability challenges to operating a webserver that can handle map data (which is frequently hundreds of megabytes in file size), and I hope that the solutions that we come up with will allow for a robust system for electron microscopy analysis on the web.</p>

  </div>

  <hr>
  <h2><a href="/2015/01/02/Welcome-2015/">Welcome to 2015</a></h2>
  <p class="author">
    <span class="author"><a href="/members#James Fraser">James Fraser</a></span><br>
    <span class="date"><em>02 January 2015</em></span>
  </p>
  <div class="content">
    <p>##Moving on up</p>

<p>We finished 2014 by moving to our new lab space up on the fourth floor of Genentech Hall. The two years we spent next to the Von Zastrow, Mostov, and Bivona Labs were absolutely fantastic. I became great friends with all the PIs (and Henry Bourne, our cluster office mate) and will miss the energy of all the students, postdocs, and staff on the floor. But, it was time to move to a labspace with a bit more room to spread out. We are excited to share equipment, space, and good times with the <a href="http://profiles.ucsf.edu/adam.frost">Frost</a>, <a href="http://systemsbiology.ucsf.edu/index.php/people">Thomson</a>, <a href="http://cryoem.ucsf.edu/">Cheng</a>, and <a href="http://bionano.ucsf.edu/">Douglas</a> labs!</p>

<p>##Photos</p>

<p>In keeping with the traditions of Genentech Hall, we removed our old lab photos on the second floor and added some new ones to the display on the fourth floor.</p>

<p><img class="img-responsive center-block" src="/static/img/news/fourth_floor.jpg" alt="Fourth Floor Photos" width="640" /></p>

<p>We went with a new theme surrounding Jennifer Li’s excellent cartoon of the lab. If you can’t tell what we are going for, you should stop by for a closer look!</p>

<p><img class="img-responsive center-block" src="/static/img/news/zoom_fourth.jpg" alt="Creepy Photo Bombs" width="640" /></p>

<p>##Rotons</p>

<p>We are welcoming Erin Poss (CCB) and Evan Green (iPQB/Biophysics) to the lab for the winter quarter. Apparently, even after <a href="/pubs">PUBS</a> they couldn’t get enough of me and Ben, who was their TA on team DUBSTEP.</p>

<p>##Preprints</p>

<p>We celebrated submitting two of Andrew’s papers at the same time as our lab-warming party. The Frost lab is new to the area too, and we are located on the same wing of our neighbourhood: the FR* wing!</p>

<p>After submitting the articles, we decided to post them on BioRXiv,  a free online archive and distribution service for unpublished preprints in the life sciences, as well. The two manuscripts can be enjoyed together and continue Andrew’s work to exploit diffuse scattering to improve macromolecular models emerging from crystallography. We had great help from collaborators on this project (and the work on the second manuscript is primarily from Sacha and Pavel). As I wrote in the cover letter:</p>

<p><a href="http://biorxiv.org/content/early/2015/01/02/012955">The first manuscript</a>, “Predicting X-ray Diffuse Scattering from Translation Libration Screw Structural Ensembles”, describes a prediction of the diffuse scattering features created by TLS refinements. When combined, the two manuscripts outline a powerful strategy for validating X-ray models that include TLS refinement. Different TLS models have equivalent agreement with Bragg data (judged by Rfree), but different agreement with diffuse data (judged by the correlation coefficients of anisotropic diffuse intensities). Our results point the way forward for using more of the recorded scattering to improve models and generate new biological insights about the role of correlated motions in protein function.</p>

<p><a href="http://biorxiv.org/content/early/2015/01/02/012930">The second manuscript</a>, “From deep TLS validation to ensembles of atomic models built from elemental motions”, describes a new tool that can convert TLS matrices to multi-model (ensemble) PDB files that in particular are required for the project described in the first manuscript. This enables easier visualization and interrogation of the correlated motions proposed by TLS refinement. This algorithm discusses that, on the top of well-known conditions that the T and L matrices must be positive definite to be interpretable by such motions, there are other conditions that are usually ignored. Ignoring these conditions creates an unusual situation: although the final ellipsoids define real atomic motions, the TLS matrices imply imaginary correlated movements of rigid bodies.</p>

<p>Happy New Year - and we look forward to more papers and preprints in 2015!</p>

  </div>

  <hr>
  <h2><a href="/2014/10/25/BBCPUBS/">BBCPUBS halfway point</a></h2>
  <p class="author">
    <span class="author"><a href="/members#James Fraser">James Fraser</a></span><br>
    <span class="date"><em>25 October 2014</em></span>
  </p>
  <div class="content">
    <div class="storify" markdown="0">

  <iframe markdown="0" src="//storify.com/fraser_lab/bbcpubs-2/embed?border=false" width="100%" height=750 frameborder=no allowtransparency=true>
    
  </iframe>

<script src="//storify.com/fraser_lab/bbcpubs-2.js?border=false"></script><noscript>[<a href="//storify.com/fraser_lab/bbcpubs-2" target="_blank">View the story "#bbcpubs" on Storify</a>]</noscript>

</div>

  </div>

  <hr>
  <h2><a href="/bk/2014/08/30/ALS-Ice-Bucket-challenge/">James takes the Ice Bucket Challenge</a></h2>
  <p class="author">
    <span class="author"><a href="/members#James Fraser">James Fraser</a></span><br>
    <span class="date"><em>30 August 2014</em></span>
  </p>
  <div class="content">
    <p>Yesterday <a href="http://web.stanford.edu/group/kimlab/">Peter Kim</a> presented me (and <a href="http://www.msg.ucsf.edu/stroud/index.htm">Bob Stroud</a> and <a href="http://www.craiklab.ucsf.edu/">Charly Craik</a>) with the <a href="http://www.alsa.org/fight-als/ice-bucket-challenge.html">Ice Bucket Challenge</a>. Peter and Susan Marqusee were challenged by <a href="https://www.youtube.com/watch?v=-ils6pggkGg">Terry Oas</a>. This fundraiser holds special significance for all of us because Tom Alber, our friend and my PhD advisor, recently passed away from ALS. I think that Tom would have loved the Ice Bucket Challenge.</p>

<div class="video-container">
         <iframe src="//www.youtube.com/embed/fQ7p9mHDJUw" frameborder="0" width="560" height="315"></iframe>
</div>
<br>
<div class="video-container">
         <iframe src="//www.youtube.com/embed/9VrzhmSkqfQ" frameborder="0" width="560" height="315"></iframe>
</div>

<p>Tom was incredibly courageous during the final years of his life. He never stopped learning about the underlying mechanisms of ALS and thinking of potential therapies. It was very painful to see him disappointed again and again by the ineffectiveness of the available treatments. Some of this disappointment is due to general difficulties in developing treatments for neurodegenerative diseases, but some of it was due to our lack of knowledge about the basic mechanisms of this disease.</p>

<div class="video-container">
         <iframe src="//www.youtube.com/embed/UJ2mUbhzWic" frameborder="0" width="560" height="315"></iframe>
</div>

<p>I challenged <a href="http://michaeleisen.org">Michael Eisen</a>, <a href="http://valelab.ucsf.edu/">Ron Vale</a>, and <a href="http://www.usfca.edu/sds/staff/">Barb Zunder</a>. Barb will help spread the message to USF. I hope that Ron will take the challenge, donate, and work to invite a top ALS researcher to create an <a href="http://www.ibiology.org/ibioseminars.html">iBioseminar</a> or <a href="http://www.ibiology.org/ibiomagazine.html">iBiomagazine</a> video about ALS. There is a real opportunity for <a href="http://www.ibiology.org/">iBiology</a> to use their production platform to capture the spirit of the moment. They can produce a fantastic video that describes the challenges and recent successes in understanding basic mechanisms of neurodegenerative disease.</p>

<p>Mike is refusing to accept the challenge to make a larger point about the relationship between funding agencies, research, and public access to research findings.  I sympathize with his point. I know that Tom was always up to date on ALS research because of the journal access afforded to him as a professor at UC Berkeley.  Other patients do not have this access and that is a shame. I hope that Mike will still donate to <a href="https://secure2.convio.net/alsa/site/Donation2;jsessionid=FB956D73A15162C5668CD3AD9D926FC3.app293b?df_id=27420&amp;27420.donation=form1">The ALS Association</a> or to a lecture <a href="http://givetocal.berkeley.edu/browse/?u=383">established in Tom’s memory at UC Berkeley</a>, or both. In the meantime, here are a few open-access papers on the subject of Amyotrophic lateral sclerosis:</p>

<ul>
  <li><a href="http://www.plosbiology.org/article/info%3Adoi%2F10.1371%2Fjournal.pbio.0060170">Protein Aggregation and Protein Instability Govern Familial Amyotrophic Lateral Sclerosis Patient Survival</a></li>
  <li><a href="http://www.plosbiology.org/article/info%3Adoi%2F10.1371%2Fjournal.pbio.1001052">A Yeast Model of FUS/TLS-Dependent Cytotoxicity</a>, by Greg Petkso (who was Tom’s PhD Advisor)</li>
  <li><a href="http://www.plosbiology.org/article/info%3Adoi%2F10.1371%2Fjournal.pbio.1000614">Molecular Determinants and Genetic Modifiers of Aggregation and Toxicity for the ALS Disease Protein FUS/TLS</a></li>
  <li><a href="http://www.plosgenetics.org/article/info%3Adoi%2F10.1371%2Fjournal.pgen.1002214">FUS and TARDBP but Not SOD1 Interact in Genetic Models of Amyotrophic Lateral Sclerosis</a></li>
  <li><a href="http://www.plosgenetics.org/article/info%3Adoi%2F10.1371%2Fjournal.pgen.1000193">Novel Mutations in TARDBP (TDP-43) in Patients with Familial Amyotrophic Lateral Sclerosis</a></li>
  <li><a href="http://www.plosgenetics.org/article/info%3Adoi%2F10.1371%2Fjournal.pgen.1000350">An ALS-Linked Mutant SOD1 Produces a Locomotor Defect Associated with Aggregation and Synaptic Dysfunction When Expressed in Neurons of Caenorhabditis elegans</a></li>
  <li><a href="http://www.plosgenetics.org/article/info%3Adoi%2F10.1371%2Fjournal.pgen.1003895">ALS-Associated FUS Mutations Result in Compromised FUS Alternative Splicing and Autoregulation</a>
-<a href="http://dx.plos.org/10.1371/journal.pone.0010064">Metal-Free ALS Variants of Dimeric Human Cu,Zn-Superoxide Dismutase Have Enhanced Populations of Monomeric Species</a>, by Bob Matthews (one of Tom’s best friends)</li>
</ul>

<p>I’ll add more links as people send or comment on them.  Fortunately, due to the hard work of people like Mike, many classic papers are now available through Pubmed Central and most journals now deposit articles after an embargo period.  In diseases like ALS where it is often only a matter of a few years between diagnosis and death, an embargo of 18 months to protect journal profits seems particular unjust.</p>

<p><strong><a href="http://givetocal.berkeley.edu/browse/?u=383">Consider making a donation to establish an infectious disease lectureship in Tom’s name at UC Berkeley</a>
and to <a href="https://secure2.convio.net/alsa/site/Donation2;jsessionid=FB956D73A15162C5668CD3AD9D926FC3.app293b?df_id=27420&amp;27420.donation=form1">The ALS Association</a></strong></p>

  </div>

<hr>

<!-- This makes links to other paginator pages -->

<ul class="pagination">
  
    <li><a href="/news/">&laquo; Prev</a></li>
  

  
    
      <li><a href="/news">1</a></li>
    
  
    
      <li class="active"><span>2</span></li>
    
  
    
      <li><a href="/news/page3">3</a></li>
    
  
    
      <li><a href="/news/page4">4</a></li>
    
  
    
      <li><a href="/news/page5">5</a></li>
    
  
    
      <li><a href="/news/page6">6</a></li>
    
  
    
      <li><a href="/news/page7">7</a></li>
    
  
    
      <li><a href="/news/page8">8</a></li>
    
  
    
      <li><a href="/news/page9">9</a></li>
    
  
    
      <li><a href="/news/page10">10</a></li>
    
  
    
      <li><a href="/news/page11">11</a></li>
    
  
    
      <li><a href="/news/page12">12</a></li>
    
  
    
      <li><a href="/news/page13">13</a></li>
    
  

  
    <li><a href="/news/page3">Next &raquo;</a></li>
  
</ul>


    </div>
    <div class="col-sm-5 col-md-4 hidden-xs">
        <a class="twitter-timeline" data-dnt="true" href="https://twitter.com/fraser_lab" data-widget-id="486191843416092672">Tweets by @fraser_lab</a>
<script>!function(d,s,id){var js,fjs=d.getElementsByTagName(s)[0],p=/^http:/.test(d.location)?'http':'https';if(!d.getElementById(id)){js=d.createElement(s);js.id=id;js.src=p+"://platform.twitter.com/widgets.js";fjs.parentNode.insertBefore(js,fjs);}}(document,"script","twitter-wjs");</script>


    </div>
</div>

    </div>
    <hr>
    <div id="footer footer-fixed-bottom">
      <div class="container">
        <p class="text-muted credit"> This website is open-source and available on <a href="https://github.com/fraser-lab/fraser-lab.github.io">Github</a>.</p>
      </div>
    </div>

     <!-- Bootstrap core JavaScript
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    
    <script src="/static/js/bootstrap.min.js"></script>
    
    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
    
      ga('create', 'UA-52773332-1', 'auto');
      ga('send', 'pageview');
    
    </script>
  </body>
</html>
