<!DOCTYPE html>

<html lang="en">
  <head>
    <meta charset="utf-8">
    <title>Macro Methods</title>
    <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no">
    <meta name="description" content="This is the official web page for the James Fraser Lab at UCSF.">
    <meta name="theme-color" content="#7fab91">
    <link href="/static/css/bootstrap.css" rel="stylesheet">
    <link href="/static/css/bootstrap-responsive.css" rel="stylesheet">
    <link href="/static/css/syntax.css" rel="stylesheet">
    <link href="/static/css/custom.css" rel="stylesheet">
    
    <script type="text/javascript" src="https://www.google.com/jsapi"></script>
    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.1/jquery.min.js"></script>
    <!-- HTML5 shim, for IE6-8 support of HTML5 elements -->
    <!--[if lt IE 9]>
      <script src="http://html5shim.googlecode.com/svn/trunk/html5.js"></script>
    <![endif]-->
  </head>

  <body>

    <!-- Static navbar -->
      <div class="navbar navbar-fixed-top navbar-default" role="navigation">
        <div class="container-fluid">
          <div class="navbar-header">
            <button type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-collapse">
              <span class="sr-only">Toggle navigation</span>
              <span class="icon-bar"></span>
              <span class="icon-bar"></span>
              <span class="icon-bar"></span>
            </button>
            <a class="navbar-brand" href="/">Fraser Lab @ UCSF</a>
          </div>
          <div class="navbar-collapse collapse">
            <ul class="nav navbar-nav navbar-right">
              
                
                  <li><a href="/" alt="home"> home</a></li>
                
              
                
                  <li><a href="/members" alt="people"> people</a></li>
                
              
                
                  <li><a href="/research" alt="projects"> projects</a></li>
                
              
                
                  <li><a href="/publications" alt="publications"> publications</a></li>
                
              
              <!-- <li><a href="https://sites.google.com/a/fraserlab.com/ucsf-private/" alt="Lab Private Area">private</a></li> -->
            </ul>
          </div><!--/.nav-collapse -->
        </div><!--/.container-fluid -->
      </div>

    <div class="container">


<h1 id="biophysics-204a-methods-in-macromolecular-structure">Biophysics 204A: Methods in Macromolecular Structure</h1>

<h2 id="fall-2017-syllabus">Fall 2017 Syllabus</h2>

<p><img src="/static/img/pub/methods_wb.JPG" alt="alt text" title="Macro methods whiteboard in the teaching lab" /></p>

<p><strong>Course Title:</strong> Methods in Macromolecular Structure</p>

<p><strong>Course Credit:</strong> 4 units</p>

<p><strong>Course Format:</strong> 12 hours of lab per week</p>

<p><strong>Location:</strong> Genentech Hall Teaching Lab - Room 227</p>

<p><strong>Prerequisites:</strong> All incoming first year BP and CCB graduate students are required to enroll in this course.</p>

<p><strong>Grading:</strong> Letter grade</p>

<p><strong>Textbook:</strong> None. Lab protocols and course materials will be available in class or online</p>

<p><strong>Course Days/Hours:</strong> Monday, Tuesday, Wednesday 1pm-5pm</p>

<p><strong>Instructors:</strong> <a href="mailto:jfraser+bp204a@fraserlab.com">James Fraser</a> and <a href="mailto:jdgross@cgl.ucsf.edu">John Gross</a></p>

<p><strong>EM Coordinator:</strong> <a href="mailto:david.bulkley@ucsf.edu">David Bulkley</a></p>

<p><strong>HSP90 Preparer/NMR guru:</strong> Ryan Tibble (Gross lab)</p>

<p><strong>HSP90 Crystallizer:</strong> Kazu Ito (Fraser lab)</p>

<p><strong>X-ray guru:</strong> Michael Thompson (Fraser lab)</p>

<p><strong>EM Computational Experts:</strong> Eugene Palovcak, Daniel Asarnow (Cheng lab)</p>

<p><strong>TAs:</strong></p>

<ul>
  <li><a href="mailto:Tak.Chio@ucsf.edu">Cynthia Chio</a></li>
  <li><a href="mailto:Andrew.Natale@ucsf.edu">Andrew Natale</a></li>
</ul>

<p><strong>Lecturers/Facilitators:</strong></p>

<p>James Fraser, John Gross, Dan Southworth, David Bulkley, James Holton, Yifan Cheng, David Agard, Aashish Manglik, Andrej Sali, Robert Stroud</p>

<p><strong>Important Dates:</strong></p>

<ul>
  <li>Method Groups Presentation: Monday, Nov 13th</li>
  <li>Break for QBC Retreat/Thanksgiving: Nov 20-25</li>
  <li>Final Teams Presentation: Monday, Dec 18</li>
</ul>

<p><strong>Background:</strong></p>

<p>Fluency in multiple biophysical methods is often critical for answering mechanistic questions. Traditionally, students are exposed to the fundamentals of multiple techniques through lectures that cover the theory prior to exposure, for some, in analysis or data collection during lab rotations. However, this structure means that only students that rotate in specific labs gain hands-on-exposure, which could limit adventurous experiments in future years.  To train the next generation of biophysicists at UCSF, we have decided to alter this traditional structure by creating a new 6 week “Macromolecular Methods” class that places data collection at the beginning of the course.  Based on our experiences designing the project-based class <a href="/pubs/">Physical Underpinnings of Biological Systems, aka PUBS!</a>, which used deep sequencing to assay the function of a comprehensive set of point mutants to introduce principles of <a href="https://www.ncbi.nlm.nih.gov/pubmed/27111525">high-throughput interrogation of biological functions</a>, we have designed Macromolecular Methods to be a team-based class where students develop their own analysis of real data that they have collected.</p>

<p><strong>Course Description:</strong></p>

<p>This is a team-based class where students work in small groups develop their own analysis of real data that they have collected. Statistical aspects of rigor and reproducibility in structural biology will be emphasized throughout lectures, journal club presentations, and hands-on activities. The course will function in three modules. In module 1 “data collection” students collect either NMR, negative stain EM, and X-ray crystallographic data. In module  2 “fundamentals of analysis”,  students will are mixed into new groups for lectures and hands-on computational tutorials. These lessons emphasize connections to both the molecular interpretations and the fundamental physical principles that generated the data. In module 3 “integrative structural biology”, the students will finalize their analysis and lectures will emphasize rigorous theory of individual techniques and computational frameworks for integrative structural modeling.  Finally, each group will present to their findings to the class and course faculty.</p>

<p><strong>Recommended reading:</strong></p>

<ul>
  <li><a href="http://science.sciencemag.org/content/339/6122/913.long">Integrative Structural Biology</a></li>
  <li><a href="https://www.ncbi.nlm.nih.gov/pubmed/8929414">SAR by NMR</a></li>
  <li><a href="http://journals.iucr.org/m/issues/2017/04/00/hi5649/index.html">Protein crystallography and drug discovery: recollections of knowledge exchange between academia and industry</a></li>
  <li><a href="http://www.pnas.org/content/105/8/3041.long">Scaffold-based design by X-ray</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/28436492">PANDDA</a></li>
  <li><a href="https://www.ncbi.nlm.nih.gov/pubmed/27238019">EM for drug discovery</a>, <a href="http://journals.iucr.org/d/issues/2017/06/00/rr5143/rr5143.pdf">Proteasome example</a></li>
  <li><a href="https://www.ncbi.nlm.nih.gov/pubmed/10468550">The maximal affinity of ligands</a></li>
  <li><a href="https://www.nature.com/articles/nrd4163">The role of ligand efficiency metrics in drug discovery</a></li>
</ul>

<p><strong>Course Goals:</strong></p>

<p>The goal of the course is to provide an immersive, hands-on experience in the context of genuine research questions. As articulated by <a href="http://www.sciencemag.org/content/338/6114/1542.long">Vale and colleagues</a>, there are tremendous advantages when graduate students work “pursuing a research question with unknown answers and uncertain outcomes, students and faculty combine their wits and skills to design experiments, evaluate progress, and troubleshoot along the way”. These advantages are likely to be common accross <a href="http://blogs.kqed.org/mindshift/2014/09/can-project-based-learning-close-gaps-in-science-education/">all learning levels</a>. In our course, teams may use whatever literature, software, and resources that are available publicly, and are encouraged to write their own scripts and software where necessary.</p>

<p>This course will introduce students to approaches and methodologies for interrogating macromolecular structure and dynamics, which will require the integration of experiment and computation. In addition to fundamental techniques in X-ray crystallography, NMR and EM, students will learn to interpret datasets, draw original conclusions, and present findings in written and oral formats.</p>

<p>The “official” language of the class is <a href="https://www.python.org">python</a> - beginners should try <a href="http://learnpythonthehardway.org/book/">Learn Python The Hard Way</a>, people with a background in other languages should try <a href="https://developers.google.com/edu/python/">Google’s python course</a>. The QB3 Berkeley <a href="http://intro-prog-bioinfo-2014.wikispaces.com/">intensive python course</a> provides many biological examples. Students should be comfortable with basic syntax and scripting prior to the start of instruction. Here is a <a href="https://docs.google.com/spreadsheets/d/1BjKsN0B1hqd4dJW5slZ5KPuToCjSMRyA7Bl8MwWrbS4/edit#gid=0">spreadsheet</a> with a listing of multiple Python resources</p>

<p><strong>Journal Clubs:</strong></p>

<p>Journal club presentations will be limited to 6 minutes. You should use 3 slides: two of which can contain a figure from the paper and the other should be a self-drawn (or created) schematic of the key concept behind the paper. Many of these papers are quite technical, so please engage the instructors and other course personnel as you prepare your presentation.</p>

<p><strong>Student Learning Objectives</strong></p>

<ul>
  <li>Laboratory safety</li>
  <li>Appropriate methods for documenting laboratory procedures</li>
  <li>Bioinformatics and algorithms</li>
  <li>Python scripting</li>
  <li>Fragment screening</li>
  <li>X-ray crystallography</li>
  <li>NMR spectroscopy</li>
  <li>Electron Miscroscopy</li>
  <li>Statistical aspects of structural biology</li>
  <li>Integrative modeling</li>
</ul>

<p><strong>Class Policies</strong></p>

<p><em>Ethics</em>: This course is more than a training experience; it is an active research project whose results will be published to the broader scientific community. The community must be able to understand our work, replicate it, and have confidence in its findings. We must therefore ensure the integrity of the information we disseminate. To do so, it is essential that students perform and document their experiments and analyses as faithfully as possible. Mistakes and oversights are normal and to be expected, but they must not be ignored, concealed, or disguised. In addition, to merit authorship, students must contribute to three aspects of the project: intellectual conception or interpretation of the methods or data, technical execution of the experiments and/or analyses, and documentation or dissemination of the results. We fully expect that by actively participating in the course and working toward the course objectives, all students will merit authorship.</p>

<p><em>Respect</em>: This course is built around an open research project performed in teams. Successful completion of the course objectives will require that students work together effectively, so please respect the time and effort of your classmates and instructors. Moreover, as part of the research process, we will consider and debate a variety of ideas and approaches; however, we must not allow our position on a particular idea or argument to compromise our respect for its author. We therefore expect course participants to give all instructors and students, regardless of academic or personal background, their complete professional respect; anything less will not be tolerated.</p>

<p><em>Absences</em>: The instructor must be notified by the second week of classes for any planned absences, or in advance of class due to illness. Active participation in the laboratory is essential and students are required to attend normal class hours. Attendance during all of the three required presentations is absolutely mandatory, except in cases of doctor-excused medical illness. Any class material or lecture that is missed will be the responsibility of the student. Written evaluations of each team and its members will be provided to the Graduate Tracking System for inclusion into the graduate record, and provided to oral committee members and thesis committee members.</p>

<p><em>Accommodations for students with disabilities:</em> The Graduate Division embraces all students, including students with documented disabilities. UCSF is committed to providing all students equal access to all of its programs, services, and activities. Student Disability Services (SDS) is the campus office that works with students who have disabilities to determine and coordinate reasonable accommodations. Students who have, or think they may have, a disability are invited to contact SDS (StudentDisability@ucsf.edu); or 415-476-6595) for a confidential discussion and to review the process for requesting accommodations in classroom and clinical settings. More information is available online
at http://sds.ucsf.edu. Accommodations are never retroactive; therefore students are encouraged to register with Student Disability Services (http://sds.ucsf.edu/) as soon as they begin their programs. UCSF encourages students to engage in support seeking behavior via all of the resources available through Student Life, for consistent support and access to their programs.</p>

<h2 id="schedule">Schedule</h2>

<p><strong>Week 1 – Welcome</strong></p>

<p><em>Tues Oct 31</em></p>

<ul>
  <li>1:00-1:30 - <a href="/static/pdf/methods/macro_methods_intro.pdf">Intro to Macro Methods</a> (JSF)</li>
  <li>1:30-2:30 - Why Hsp90 is cool (David Agard)</li>
</ul>

<p>Chalk Talks:</p>

<ul>
  <li>2:30-2:45 - Why Protein Structural Biology? Proteins Thermo/Kinetics 101 (Gross)</li>
  <li>2:45-3:00 - X-ray 101 (JSF)</li>
  <li>3:00-3:15 - EM 101 (Dan Southworth)</li>
  <li>3:15-3:30 - NMR 101 (Gross)</li>
</ul>

<p><em>Weds Nov 1</em></p>

<ul>
  <li>1:00-2:00PM - break into groups, ensure software works
    <ul>
      <li>All:
        <ul>
          <li><a href="https://www.cgl.ucsf.edu/chimerax/download.html">ChimeraX</a></li>
          <li><a href="https://pymol.org/2/">PyMOL</a>, must be on UCSF network</li>
        </ul>
      </li>
      <li>X-ray:
        <ul>
          <li><a href="https://www.phenix-online.org/download/">Phenix</a>, <a href="http://www.phenix-online.org/phenix_request/index.cgi">request password</a></li>
          <li><a href="http://xds.mpimf-heidelberg.mpg.de/html_doc/downloading.html">XDS</a></li>
          <li><a href="https://strucbio.biologie.uni-konstanz.de/xdswiki/index.php/Installation#Mac_.28Intel.29">XDS-GUI</a></li>
          <li><a href="http://www.scripps.edu/tainer/arvai/adxv.html">ADXV</a></li>
          <li>Coot (already installed on laptops), if not use <a href="http://scottlab.ucsc.edu/xtal/wiki/index.php/Stand-alone_Coot_for_10.11">Standalone Coot</a></li>
          <li><a href="http://www.phillipslab.org/software">XRayView</a></li>
        </ul>
      </li>
      <li>NMR:
        <ul>
          <li><a href="https://www.ibbr.umd.edu/nmrpipe/install.html">NMR Pipe</a>, <a href="/static/pdf/methods/NMRPipe_Install_MacOS.pdf">MacOSX Install Guide</a></li>
          <li><a href="http://www.nmrfam.wisc.edu/nmrfam-sparky-distribution.htm">NMRFAM SPARKY</a></li>
        </ul>
      </li>
      <li>EM:
        <ul>
          <li><a href="http://ncmi.bcm.tmc.edu/ncmi/software/software_details?selected_software=counter_222">EMAN2</a></li>
        </ul>
      </li>
    </ul>
  </li>
  <li>2:00-3:00PM - <a href="/static/pdf/methods/20171029_Fragments_Lecture.pdf">Theory of Fragments and Role of Structural Biology</a> (Aashish Manglik)</li>
</ul>

<p><strong>Week 2 – Working in Method Teams</strong></p>

<p><em>X-ray</em></p>
<ul>
  <li>Christopher John Pascal	Mathy, Yessica	Gomez, Colton	Bracken, Conner	Bardine, Adam	Cotton, Paul	Klauser, Lakshmi	Miller-Vedam</li>
  <li>Monday Nov 6: Crystal harvesting (Genentech Hall crystal room)</li>
  <li>Tuesday Nov 7: Beamtime at Advanced Light Source: noon-4PM
    <ul>
      <li><a href="https://docs.google.com/document/d/1V2-PNSKaoAF7rlRj-rJNA9xA8E-7zrwWzigpKhcNNDI/edit?usp=sharing">link to registration material</a> - register at <a href="alshub.als.lbl.gov">ALSHub</a></li>
      <li><a href="https://als.lbl.gov/onsite-access/">directions to the ALS</a>, we will travel together on BART Nov 7 and 8, but just in case!</li>
    </ul>
  </li>
  <li>Wednesday Nov 8: Beamtime at Advanced Light Source: 8AM-4PM</li>
</ul>

<p><em>NMR</em></p>
<ul>
  <li>NMR Group A:  Viktoriya	Berdan, Adam	Catching, Neha	Prasad, Jack	Stevenson</li>
  <li>NMR Group B:  Nicole	Wenzell, Eric	Gonzalez, Cody Thomas	Krivacic</li>
  <li>Monday Nov 6:	  1-2pm NMR sample loading (teaching lab, Group A and B); NMR data collection: 2-3:30pm, Group A and 3:30-5pm , Group B</li>
  <li>Tuesday Nov 7:	  1-5pm NMR data collection (titration, Group A )</li>
  <li>Wednesday Nov 8:	   1-5pm NMR data collection (titration, Group B)</li>
  <li><a href="/static/pdf/methods/UCSF_500_Cheat_Sheet.pdf">500 MHz NMR quick start guide</a></li>
</ul>

<p><em>EM</em></p>
<ul>
  <li>Kyle	Lopez, Erik	Navarro, Paige	Solomon, Kelly	Montgomery, Jenna	Pellegrino, Megan	Moore, Julian	Harris</li>
  <li>Monday Nov 6: Grids</li>
  <li>Tuesday Nov 7: Negative Stain Data Collection</li>
  <li>Wednesday Nov 8: Cryo-EM</li>
</ul>

<p><strong>Week 3 – Working in Compound Groups</strong></p>

<p><em>Groups</em></p>
<ul>
  <li>1: Erik	Navarro, Paige	Solomon, Adam	Catching, Neha	Prasad, Christopher John Pascal	Mathy</li>
  <li>2: Kelly	Montgomery, Jack	Stevenson, Yessica	Gomez, Colton	Bracken</li>
  <li>3: Jenna	Pellegrino, Nicole	Wenzell, Conner	Bardine, Adam	Cotton</li>
  <li>4: Kyle	Lopez, Viktoriya	Berdan, Eric	Gonzalez, Paul	Klauser</li>
  <li>5: Megan	Moore, Julian	Harris, Cody Thomas	Krivacic, Lakshmi	Miller-Vedam</li>
</ul>

<p><em>Mon Nov 13</em>
<strong>Presentations on Methods Week</strong>: Two students from each Method Team will present for 10 minutes and summarize what occured in Week 2. Take pictures and try to give the students a feel for not only the theory of what you learned but also the practical aspects!</p>
<ul>
  <li>EM: Erik	Navarro and Paige	Solomon</li>
  <li>X-ray: Christopher John Pascal	Mathy and Yessica	Gomez</li>
  <li>
    <p>NMR: Adam	Catching and Neha	Prasad</p>
  </li>
  <li>1-2:20PM: X-ray Methods Group Presentation and X-ray data processing: from spots to MTZ (JSF, Kazu Ito, Michael Thompson)
    <ul>
      <li><a href="https://ucsf.box.com/s/i85gxe5prx9z3ov5hdftqgy1skdevca7">Box link to ADP images</a> - Download</li>
      <li><a href="/static/apo/xia2.html">xia2 summary for APO</a>
        <ul>
          <li><a href="/static/apo/DataFiles/yHSP90_APO.mtz">APO mtz</a></li>
        </ul>
      </li>
      <li><a href="/static/adp/xia2.html">xia2 summary for ADP</a>
        <ul>
          <li><a href="/static/adp/DataFiles/yHSP90_ADP.mtz">ADP mtz</a></li>
        </ul>
      </li>
    </ul>
  </li>
  <li>2:20-3:40PM: EM Methods Group Presentation and <a href="https://docs.google.com/document/d/1cyh9XstW3Da8tU1g5gj8UCk1OMC8vGJsk7a5r6T9XHo/edit?usp=sharing">EM data processing: from particles to 2D</a></li>
  <li>3:40-5PM NMR Methods Group Presentation and NMR data processing: : <a href="/static/pdf/methods/2DNMR_Processing_Tutorial.pdf">from FID to 2D</a> (Ryan Tibble and John Gross)</li>
</ul>

<p><em>Tues Nov 14</em></p>
<ul>
  <li>1-2PM <a href="https://docs.google.com/presentation/d/1RXALgbFH0JQmjFozPzkO8Oe2OenqU4D4Nl7md6UBMP0/edit?usp=sharing">EM - FFT 101</a></li>
  <li>2-3PM NMR data processing: overlaying spectrum with assignments and perturbations from 2D data comparisons (Ryan Tibble and John Gross)
    <ul>
      <li><a href="/static/pdf/methods/Sparky_tutorial.pdf">Intro to Sparky</a></li>
      <li>Compound Assignments (25=DMSO, 26=APO):
        <ul>
          <li>1: 1, 2, 11, 13, 19</li>
          <li>2: 3, 4, 12, 14, 21</li>
          <li>3: 5, 6, 15, 17, 22</li>
          <li>4: 7, 8, 16, 20, 24</li>
          <li>5: 9, 10, 18, 23</li>
        </ul>
      </li>
    </ul>
  </li>
  <li>3PM - Journal club:
    <ul>
      <li>Nicole	Wenzell <a href="https://doi.org/10.1016/j.sbi.2005.08.006">Weak alignment NMR: a hawk-eyed view of biomolecular structure</a>, background: <a href="https://www.ncbi.nlm.nih.gov/pubmed/9353189">Direct measurement of distances and angles in biomolecules by NMR dilute liquid crystalline medium</a></li>
      <li>Lakshmi	Miller-Vedam <a href="https://www.nature.com/nature/journal/v355/n6359/pdf/355472a0.pdf">Free R value: a novel statistical quantity for assessing the accuracy of crystal structures</a></li>
    </ul>
  </li>
  <li>3:30-4:30PM X-ray data processing: from MTZ to density, molecular replacement and difference maps (JSF, Kazu Ito, Michael Thompson)</li>
</ul>

<p><em>Weds Nov 15</em></p>
<ul>
  <li>1-2PM NMR data processing: measuring chemical shift perturbations  (Ryan Tibble and John Gross)</li>
  <li>2-3PM X-ray data processing: identifying ligands, ligand restraints and refinement (JSF, Kazu Ito, Michael Thompson)</li>
  <li>3PM - Journal Club:
    <ul>
      <li>Kelly	Montgomery <a href="https://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=22842542">Prevention of overfitting in cryo-EM structure determination.</a>, background: <a href="http://www.sciencedirect.com/science/article/pii/S0022283603010222">Optimal Determination of Particle Orientation, Absolute Hand, and Contrast Loss in Single-particle Electron Cryomicroscopy</a></li>
      <li>Kyle	Lopez <a href="https://www.ncbi.nlm.nih.gov/pubmed/20057044">Molprobity</a></li>
    </ul>
  </li>
  <li>3:30-4:30PM <a href="https://docs.google.com/document/d/1cyh9XstW3Da8tU1g5gj8UCk1OMC8vGJsk7a5r6T9XHo/edit?usp=sharing">EM data processing: manipulating density</a>
    <ul>
      <li><a href="https://ucsf.box.com/s/aoz3ydxn9rdvowxhcbfm6qbmo159x3rv">PDBs</a> for FSC comparisons</li>
    </ul>
  </li>
</ul>

<p><strong>Thanksgiving break</strong></p>

<p><strong>Week 4 - NMR</strong></p>

<p><em>Mon Nov 27</em></p>
<ul>
  <li>Journal Club:
    <ul>
      <li>Jack	Stevenson <a href="https://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=20159554">Asymmetric activation of the hsp90 dimer by its cochaperone aha1.</a>, background: <a href="https://www.ncbi.nlm.nih.gov/pubmed/8946851">Localization of the NADPH binding site on MurB by NMR</a></li>
    </ul>
  </li>
  <li>1-2:30 Lecture by John Gross on fundamentals of NMR theory applications to  mapping protein interactions binding, <a href="/static/pdf/methods/Methods-NMR-l2017.pdf">NMR lecture notes</a></li>
  <li>2:30-5pm work on analyzing NMR titration data to determine Kd (Ryan Tibble and John Gross), <a href="/static/pdf/methods/Sparky_common_cmds.pdf">Common Sparky Commands</a> and <a href="/static/pdf/methods/Sparky_Integration.pdf">Sparky Integration Tutorial</a></li>
</ul>

<p><em>Tues Nov 28</em></p>
<ul>
  <li>Journal Club:
    <ul>
      <li>Viktoriya	Berdan  <a href="https://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=21474071">Substrate binding drives large-scale conformational changes in the Hsp90 molecular chaperone.</a></li>
      <li>Cody Thomas Krivacic <a href="https://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=22063096">Cross-monomer substrate contacts reposition the Hsp90 N-terminal domain and prime the chaperone activity.</a></li>
    </ul>
  </li>
  <li>1-2:30 Lecture by John Gross on using NMR for structure determination.</li>
  <li>2:30-5pm work on generating chemical shift perturbation plot versus primary sequence; mapping onto structure, <a href="/static/pdf/methods/Sparky_CSP.py">CSPs from Sparky Lists</a></li>
</ul>

<p><em>Weds Nov 29</em></p>
<ul>
  <li>Journal Club:
    <ul>
      <li>Eric	Gonzalez <a href="https://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=28218749">Mechanistic basis for the recognition of a misfolded protein by the molecular chaperone Hsp90.</a></li>
    </ul>
  </li>
  <li>1-2:30 Lecture by John Gross on using NMR to measure ms-usec dynamics in proteins (Ryan Tibble and John Gross)</li>
  <li>2:30-5pm mapping NMR shift perturbations onto structure using Chimera  (John Gross, Ryan Tibble, Tom Goddard)</li>
  <li><a href="http://www.cgl.ucsf.edu/chimera/data/nmrcolor-nov2017/nmrcolor.html">ChimeraX tutorial by Tom Goddard</a></li>
  <li><a href="/static/pdf/methods/Macro_NMR_Screen_Cmpd_Structures.pdf">NMR Screen Compound Structure Gallery</a></li>
</ul>

<p><strong>Week 5 - X-ray</strong></p>
<ul>
  <li>
    <p>Compound Data Processing Logs, MTZs, CIFs, PDBs:</p>

    <ul>
      <li>1 : a50 a51 c1 c11 c13 c2 f1 f14 f19 f23 f28 f32 f37 f41 f46 f51 f58 f63 f68 f73 f8 f85 f9 f95 - <a href="https://ucsf.box.com/s/nh3euf7zj9wvma40tm6l7n64e6zdnvk1">BOX LINK</a></li>
      <li>2 : amppnp c12 c14 c3 c4 f10 f15 f2 f24 f29 f33 f38 f42 f47 f52 f59 f64 f69 f75 f80 f86 f90 f96 - <a href="https://ucsf.box.com/s/6m2g3xyaxcudy50ib47011baraf1ywd3">BOX LINK</a></li>
      <li>3 : a40 c15 c22 c5 c6 f11 f16 f20 f25 f3 f34 f39 f43 f48 f53 f55 f56 f6  f7 f76 f81 f91 f92 - <a href="https://ucsf.box.com/s/eifn44wo5ye00kcvyk728yfe120l9qxf">BOX LINK</a></li>
      <li>4 :  a44 c8 f12 f17 f21 f26 f30 f35 f4 f44 f49 f54 f60 f65 f66 f70 f71 f77 f77_2 f78 f83 f88 f93 - <a href="https://ucsf.box.com/s/0tr7age0396ksiqnblvi3ij7jqxwdns7">BOX LINK</a></li>
      <li>5 : c10 c23 c9 f13 f18 f22 f27 f31 f36 f40 f45 f50 f57 f61 f67 f72 f74 f79 f82 f84 f87 f89 f94  - <a href="https://ucsf.box.com/s/cq66o2i445ngbdnvne5gblslt8vii6yk">BOX LINK</a></li>
    </ul>
  </li>
  <li>
    <p><a href="https://docs.google.com/spreadsheets/d/1a-eGuT2_d_yWOlCJEn-Oey-0_t-kFF5cl7FuAeOwQkw/edit?usp=sharing">SMILES strings for all compounds</a></p>
  </li>
  <li>
    <p><a href="https://docs.google.com/document/d/1eQfOtSrB2gyF1BJQYWLI-mSoYsENyg4xP8Fsi5ZqsfM/edit?usp=sharing">Some advice on how to proceed</a></p>
  </li>
</ul>

<p><em>Mon Dec 4</em></p>
<ul>
  <li>1-2:30PM <a href="https://drive.google.com/file/d/14gjFiRS2KLdZ8GqfAlrqZIbzH08gZ9TT/view?usp=sharing">Lecture by Bob Stroud on fundamentals of X-ray diffraction</a></li>
  <li>2:30-5PM Work on X-ray data (JSF, Kazu Ito, Michael Thompson)</li>
  <li>4PM - Journal Club:
    <ul>
      <li>Colton	Bracken <a href="http://www.sciencedirect.com/science/article/pii/S0969212616301356?via%3Dihub">High-Throughput Crystallography: Reliable and Efficient Identification of Fragment Hits</a></li>
    </ul>
  </li>
</ul>

<p><em>Tues Dec 5</em></p>
<ul>
  <li>1-2:30PM <a href="http://bl831.als.lbl.gov/~jamesh/powerpoint/UCSF_DQvMI_2017.pptx">Lecture by James Holton on the relationship between data quality and model interpretation</a></li>
  <li>2:30-5PM Work on X-ray data (JSF, Kazu Ito, Michael Thompson)</li>
  <li>4PM - Journal Club:
    <ul>
      <li>Conner	Bardine <a href="https://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=23251785">Modelling dynamics in protein crystal structures by ensemble refinement</a> and</li>
      <li>Adam	Cotton <a href="https://www.ncbi.nlm.nih.gov/pubmed/8846217">Why protein crystals favour some space-groups over others.</a></li>
    </ul>
  </li>
</ul>

<p><em>Weds Dec 6</em></p>
<ul>
  <li>1-2PM Presentation by Sali on Rigor, Reproducibility, and Integrative Modeling</li>
  <li>2:30-4PM Final work on X-ray data (JSF, Kazu Ito, Michael Thompson)</li>
  <li>4PM - Journal Club:
    <ul>
      <li>Paul	Klauser <a href="https://www.ncbi.nlm.nih.gov/pubmed/28742020">Symmetry broken and rebroken during the ATP hydrolysis cycle of the mitochondrial Hsp90 TRAP1.</a></li>
    </ul>
  </li>
  <li>4-5PM Final Q/A with Stroud and Fraser: what we still don’t understand about X-ray crystallography</li>
</ul>

<p><strong>Week 6 - EM</strong></p>

<p><em>Mon Dec 11</em></p>
<ul>
  <li>1-2:30PM <a href="https://drive.google.com/open?id=1lZWQIS1Yjm8hZuNxXQbddmBKhpIQSu9k">Lecture by Yifan Cheng</a></li>
  <li>2:30-5PM Work on <a href="https://ucsf.box.com/s/5miwi4zdzuxagx0b9x6mekkaby9v3gh1">EM data</a></li>
  <li>4PM - Journal Club:
    <ul>
      <li>Jenna	Pellegrino <a href="https://www.ncbi.nlm.nih.gov/pubmed/24949977">Glucocorticoid receptor function regulated by coordinated action of the Hsp90 and Hsp70 chaperone cycles</a></li>
    </ul>
  </li>
  <li><a href="http://www.cgl.ucsf.edu/chimera/data/hsp90-em-dec2017/hsp90em.html">ChimeraX EM tutorial by Tom Goddard</a></li>
  <li><a href="http://www.cgl.ucsf.edu/chimera/data/hsp90-ligands-dec2017/ligandview.html">ChimeraX Ligand Comparison tutorial by Tom Goddard</a></li>
</ul>

<p><em>Tues Dec 12</em></p>
<ul>
  <li>1-2:30PM <a href="https://drive.google.com/open?id=1lZWQIS1Yjm8hZuNxXQbddmBKhpIQSu9k">Lecture by Yifan Cheng</a></li>
  <li>4PM - Journal Club:
    <ul>
      <li>Megan	Moore <a href="https://www.biorxiv.org/content/early/2017/10/23/207928">Visualization of ligand-induced transmembrane signalling in the full-length human insulin receptor</a></li>
    </ul>
  </li>
  <li>2:30-5PM Work on projection matching with Eugene/Daniel</li>
</ul>

<p><em>Weds Dec 13</em></p>
<ul>
  <li>1-2PM <a href="https://drive.google.com/open?id=1lZWQIS1Yjm8hZuNxXQbddmBKhpIQSu9k">Lecture by Yifan Cheng</a></li>
  <li>4PM - Journal Club:
    <ul>
      <li>Julian	Harris <a href="https://www.ncbi.nlm.nih.gov/pubmed/28619716">Ratchet-like polypeptide translocation mechanism of the AAA+ disaggregase Hsp104.</a></li>
    </ul>
  </li>
  <li>2:30-5PM Finalize work on all projects!</li>
</ul>

<p><strong>FINAL PRESENTATIONS: Mon Dec 18</strong></p>

<p>Please be on time and wait outside the teaching lab before your presentation. The presentations will be stopped after 15 min and questions will be for 5 minutes. Please email your presentations (use a filename that includes your team number!) to <a href="mailto:jfraser+bp204a@fraserlab.com">James Fraser</a> by 12:30PM on Monday Dec 18th.</p>

<ul>
  <li>1-1:20 TEAM 1: Erik	Navarro, Paige	Solomon, Adam	Catching, Neha	Prasad, Christopher John Pascal	Mathy</li>
  <li>1:25-1:45 TEAM 2: Kelly	Montgomery, Jack	Stevenson, Yessica	Gomez, Colton	Bracken</li>
  <li>1:50-2:10 TEAM 3: Jenna	Pellegrino, Nicole	Wenzell, Conner	Bardine, Adam	Cotton</li>
  <li>2:15-2:35 TEAM 4: Kyle	Lopez, Viktoriya	Berdan, Eric	Gonzalez, Paul	Klauser</li>
  <li>2:40-3:00 TEAM 5: Megan	Moore, Julian	Harris, Cody Thomas	Krivacic, Lakshmi	Miller-Vedam</li>
</ul>


    </div>
    <hr>
    <div id="footer footer-fixed-bottom">
      <div class="container">
        <p class="text-muted credit"> This website is made using the template of <a href="https://github.com/fraser-lab/fraser-lab.github.io">the Fraser Lab</a>.</p>
      </div>
    </div>

     <!-- Bootstrap core JavaScript
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    
    <script src="/static/js/bootstrap.min.js"></script>
    
    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
    
      ga('create', 'UA-52773332-1', 'auto');
      ga('send', 'pageview');
    
    </script>
  </body>
</html>
